\section{Methods and Data} \label{s:lit:computational}

\vspace{3mm}
% \noindent\rule{17cm}{0.2pt}
\fbox {
    \parbox{\linewidth}{
      \begin{itemize}
        \item Datasets used: \acrlong{tcga} and non-tumour from \acrlong{jbu}
        \item Clustering methods and the metrics used
        \item Networks and describing a network
        \item Tools for statistical analysis of gene expression
      \end{itemize}
    }
}
\vspace{3mm}

% Introduction to the sections
In the \acrfull{ml} field there are three types of learning: supervised, semi-supervised and unsupervised learning. The first case is when the human labels the data, the corollary being that there is a need for careful processing as well as already having information about the data. This is the 'easiest' case as it comes with a wealth of information and the output is known to belong to the pre-defined set of labels. In supervised learning are the most recent progress in \acrfull{dl} was made but it is usually not a realistic scenario as labelling data is usually challenging and costly. The semi-supervised is when the model is not given labelled data to learn from, but a set of rules (rewards/punishment) from where it needs to find the solution. These types of models are applied to problems which can be solved by trial and error, and being able to try different learning strategies.

% Introduce the unsupervised
This project uses unsupervised approaches to analyse, learn and ultimately find new groups of \acrfull{mibc}. This type of ML methods are applied to problems where there are not defined patterns, classes and the outcome is not clear. Clustering is the main approach to analyse the data, to find the present patterns (or groups) which  are then validated with domain knowledge or by experts in the fields. 

% Talk about the data characteristics
From a data science perspective, the the omics datasets are characterised by having a small number of samples with a relative large number of features, making it challenging to apply the supervised and semi-supervised learning algorithms. Especially on the disease stratification, unsupervised learning is more suitable as it is used to discover new patterns in the biological data.

This part of the thesis serves as a foundational part for all the results chapters, where various techniques were employed to assess clustering configurations, evaluate the biological significance of \acrshort{mibc}, and define its molecular characteristics. The chapter starts covering the datasets used in the project (\cref{s:lit:datasets_used}), followed by introducing the clustering models (\cref{s:lit:clustering}), the metrics used to assess their performance (\cref{s:lit:clustering}), and dimension reduction methods (\cref{s:lit:dim_red}). It also introduces the concepts of networks/graphs in \cref{s:lit:graph_overview} and the methods for describing them in \cref{s:lit:net_metrics}. The main tools for analysing MIBC subgroups include \acrfull{dea}, which incorporates Volcano and Pi plots (\cref{s:lit:dea,s:lit:pi}), and \acrfull{gsea} (\cref{s:lit:gsea}). Finally, Kaplan-Meier plots are introduced in \cref{s:lit:survival}.



\import{Sections/Lit_review/Methods/}{datasets.tex}

\import{Sections/Lit_review/Methods/}{computational.tex}

\import{Sections/Lit_review/Methods/}{networks.tex}

\import{Sections/Lit_review/Methods/}{biological_analysis.tex}

% \import{Sections/Lit_review/Yr1/}{biomedical.tex}
% \import{Sections/Lit_review/Yr1/}{other_methpds.tex}


\subsection{Summary} \label{s:lit:choosing_ml}


This section covered the tools used to analyse the gene expression datasets in this project. It is important to note that this section was not a literature review of these techniques but rather a background for the methods used throughout the project. \Cref{s:lit:multi-omics} covers different approaches to analysing omics data, \cref{s:lit:nets_bio} presents research on using networks within biology, and \cref{s:lit:comm_detect} focuses on detecting groups within networks.

From an engineering perspective, the problem studied in this project is characterised by a large number of features and a small number of samples. It is an unsupervised learning type problem, which needs to determined the groups in an unlabelled data. There are multiple levels of information available for each gene which includes gene expression, mutations and epigenetic data. Therefore, it is important to choose a computational method that satisfies the project's aims and objectives to integrate multiple data types. The next section aims to review the work done across the genomics field by examining various approaches depending on the datasets used.
