\section{Methods and Data} \label{s:lit:computational}

\vspace{3mm}
% \noindent\rule{17cm}{0.2pt}
\fbox {
    \parbox{\linewidth}{
      \begin{itemize}
        \item Datasets used: \acrlong{tcga} and non-tumour from \acrlong{jbu}
        \item Clustering methods and the metrics used
        \item Networks and describing a network
        \item Tools for statistical analysis of gene expression
      \end{itemize}
    }
}
\vspace{3mm}

% Briefly mentions the other types of learning and why are not suitable
In \acrfull{ml} there are three types of learning: supervised, semi-supervised and unsupervised learning. The first case is when the human annotates the data, the corollary being that there is a need for careful processing and already holding information about the data. This is the 'easiest' case as it comes with a wealth of information and the output is known to belong to one of the pre-defined classes; the challenges come from labelling the dataset. The semi-supervised is when the model is not provided labelled data to learn from, but a set of rules (rewards/punishment) from where it needs to find the solution. These models are applied to problems that can be solved by trial and error, and where learning strategies can be defined. 

The omics datasets are characterised by having a small number of samples with a relative large number of features, making it challenging to apply the supervised or semi-supervised learning algorithms as there is insufficient data or the rules are hard to define. Especially on the disease stratification, unsupervised learning is more suitable as it is used to discover new patterns in the biological data. Therefore, this project uses unsupervised approaches to analyse, learn and ultimately find new groups of \acrlong{mibc}. This type of ML methods are applied to problems without defined patterns, classes and the outcome is not clear. Clustering is the main approach to analyse the data, to find the patterns (or groups) which  are then validated with domain knowledge or by experts in the fields. 

This part of the thesis serves as a foundation for all the results chapters, where various techniques were employed to assess clustering configurations, evaluate the biological significance of \acrshort{mibc}, and define its molecular characteristics. The chapter starts by covering the datasets used in the project (\cref{s:lit:datasets_used}), followed by introducing the clustering models (\cref{s:lit:clustering}), the metrics used to assess their performance (\cref{s:lit:clustering}), and dimension reduction techniques (\cref{s:lit:dim_red}). It also explains the concepts of networks/graphs in \cref{s:lit:graph_overview} and the methods for describing them in \cref{s:lit:net_metrics}. The \cref{s:lit:gene_analysis} covers the main main tools for analysing MIBC subgroups including \acrfull{dea}, which contains the Volcano and Pi plots (\cref{s:lit:dea,s:lit:pi}), \acrfull{gsea} (\cref{s:lit:gsea}) and Kaplan-Meier in \cref{s:lit:survival}.



\import{Sections/Lit_review/Methods/}{datasets.tex}

\import{Sections/Lit_review/Methods/}{computational.tex}

\import{Sections/Lit_review/Methods/}{networks.tex}

\import{Sections/Lit_review/Methods/}{biological_analysis.tex}

% \import{Sections/Lit_review/Yr1/}{biomedical.tex}
% \import{Sections/Lit_review/Yr1/}{other_methpds.tex}


\subsection{Summary} \label{s:lit:choosing_ml}


This section covered the tools used to analyse the gene expression datasets in this project. It is important to note that this section was not a literature review of these techniques but rather a background for the methods used throughout the project. \Cref{s:lit:multi-omics} covers different approaches to analysing omics data, \cref{s:lit:nets_bio} presents research on using networks within biology, and \cref{s:lit:comm_detect} focuses on detecting groups within networks.

From an engineering perspective, the problem studied in this project is characterised by a large number of features and a small number of samples. It is an unsupervised learning type problem, which needs to determined the groups in an unlabelled data. There are multiple levels of information available for each gene which includes gene expression, mutations and epigenetic data. Therefore, it is important to choose a computational method that satisfies the project's aims and objectives to integrate multiple data types. The next section aims to review the work done across the genomics field by examining various approaches depending on the datasets used.
