\section{A network approach to subtype MIBC}


\vspace{3mm}
\fbox{
    \parbox{0.95\linewidth}{
      \begin{itemize}
        \item First attempt to integrate mutations and Transcription Factors
        \item Use non-cancerous data (P0) to inform MIBC subtyping 
        \item Develop a network pipeline (inspired from PGCNA) and tools to analyse results
      \end{itemize}
    }
}
\vspace{3mm}


%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\subsection{Summary}

In the previous chapter, standard clustering methods and insights from in situ experiments (\citet{Baker2022-bj}) were utilised to stratify the muscle-invasive bladder cancer (MIBC) cohort from The Cancer Genome Atlas (TCGA). This work uncovered new MIBC subgroups with significant biological differences compared to those identified in the TCGA (\citet{Robertson2017-mg}) and consensus (\citet{Kamoun2020-tj}) stratification. This finding highlights the potential for discovering biologically relevant groups and underscores the need for new methods that combine data at the computational level.

Networks/Graphs methods have been previously used to stratify other cancers (glioblastoma - \citet{Care2019-ij}) and the gene expression data is represented in a way that enables further information integration.  Additionally, networks serve as powerful visualisation tools, enhancing the biological understanding of diseases.


This chapter delves into how mutations and transcription factors can be integrated into a network approach, aiming to uncover new MIBC subgroups. Two primary networks are constructed: one from the tumour dataset (Section \ref{s:N_I:tum}) and another from the healthy dataset (Section \ref{s:p0}). The experiments with the tumour network aim to demonstrate the potential of the network approach and to draw comparisons with the findings from the previous chapter. Conversely, the experiments with the healthy network investigate how non-cancerous information can inform MIBC stratification.

Furthermore, this chapter focuses on understanding how different parameters in the network pipeline influence the graphs and disease subtyping. It involves developing tools and methods to analyse the vast amount of results, highlighting the challenges inherent in working with unsupervised learning (in machine learning terminology), particularly in the context of highly heterogeneous data.

Overall, the work presented in this section demonstrates the potential of networks in MIBC subtyping. Although the initial methods for data integration show limited impact on disease subtyping, this chapter lays the groundwork for future chapters, where initial assumptions are revised, and new, adapted methods are employed.

\subsubsection{Datasets}

The MIBC cohort from The Cancer Genome Atlas \cite{Robertson2017-mg} and the P0 samples from Jack Birch Unit (JBU) are used. The first dataset is used in the first part of the chapter where a network is created from the genes expressed in the tumours and then subtype the cohort. While in second part, the network is constructed from the genes expressed in the P0 samples which are then used to subgroup the MIBC cohort from TCGA. Both datasets consists of Transcriptions Per Million (TPM) from RNA sequencing (RNAseq). 

The mutation information is coming from Whole Exon Sequencing (WES) of the MIBC cohort from TCGA. In this chapter only the mutation count, i.e. how much a gene is mutated across the cohort. In the following chapters the mutation type (e.g. if it is missense) is used to integrate more details.

The list of Transcription Factors (TFs) is taken from The Human Transcription Factors work \citet{Lambert2018-el}.



% This needs to be in the literature review side of things
% \import{Sections/Network_I/}{review.tex}


\import{Sections/Network_I/}{methods.tex}
\newpage
\import{Sections/Network_I/}{tum.tex}
\newpage
\import{Sections/Network_I/}{p0.tex}
\newpage
\import{Sections/Network_I/}{selective_pruning.tex}
\newpage

\subsection{Discussion}

This chapter presents the initial attempt of this project to apply a network-based approach for stratifying the muscle invasive bladder cancer (MIBC) cohort from the TCGA database. The approach involved integrating multiple data types at the computational stage, including gene expression from both cancerous and non-cancerous tissues, mutation burden, and transcription factors (TFs). The integration occurred at different levels, starting by building a graph from the non-tumour data and then using the representation to derive the MIBC subtypes, as described in \cref{s:N_I:p0}. The edge weights derived from the Spearman correlation are modified based on the mutation burden, and the node degree is limited depending on whether a gene is a transcription factor or not.



% Weight modifiers need refinement
Two main co-expressed gene networks were explored: one constructed from tumour samples and the other from healthy tissue samples. The experiments demonstrated that integrating mutations and TFs was successful at the graph level but had limited impact on the MIBC subtypes derived from the network. Network metrics varied between the modified networks (as shown in Figures \ref{fig:N_I:net_metrics_tum} and \ref{fig:N_I:net_metrics_p0}), and highly mutated genes were observed to change communities (Figure \ref{fig:N_I:p0_mut_burden}). The disease subgroups identified through this network approach showed different results compared to standard approaches, with only minor variations observed between networks modified by weight modifiers. This suggests that the weight modifier strategies, as depicted in Figure \ref{fig:N_I:modifiers}, require further refinement.

% P0 not enough
The networks built from the P0 (\cref{s:N_I:p0}) highlighted the problems introduced to the Leiden algorithm when a subset of genes is allowed a larger number of connections (50). The P0 network is also built on an incomplete representation of the MIBC tumours, as there is no molecular representation of the basal group (undifferentiated). Additionally, the gene expression of the P0 samples is not considered when subtyping the MIBC. These limitations are addressed in the next chapter, where the entire available non-tumour dataset is used and a new version of MEV is introduced.


% Community detection
Varying the number of edges per TF significantly impacts the community detection algorithms, Leiden more than SBM. It was observed that the former finds fewer communities as the nodes' degree is increased, while SBM is more stable. This, along with the work by \citet{Peixoto2021-jx, Peixoto2023-rt}, which shows that Modularity Maximisation methods are prone to finding patterns in noise, made SBM the preferred method in this project. In this chapter, the simple degree-corrected version of SBM was used, but a hierarchical version of it will be used in the next chapter. The hierarchical SBM is more complex and computationally expensive than the standard version, but it does not have a threshold on the number of communities it can find.


% Edge pruning successful - decided on the 6TF. also mentioned that the controls used enabled us to find relevant 98 TFs
All the experiments contributed to advancing the network pipeline to the next iteration in the next chapter, but the selective edge pruning to integrate TF knowledge arguably has the highest biological impact. Through the control experiments, it was observed that there are 98 TFs that are 'naturally' highly connected. The list of genes was further analysed and it was found that there is an MIBC subgroup with a very poor prognosis. The work done in \cref{s:N_I:sel_tfs} shows the impact of changing the number of connections, demonstrating that the edge pruning strategy can lead to relevant biological findings. It also shows how the network can be a useful tool for biologists by selecting sub-graphs based on the genes under study.


% Next section
This chapter explored three types of data integration in a graph: at a network level, edge weights, and the number of connections. The first two strategies need further refining, while for the selective edge pruning, a minimum degree of 6 for TFs and 3 for standard genes was found to be appropriate. The subsequent section will consider the following changes: 1) utilising the entire non-tumour dataset to enhance gene representation in the tumour network; 2) using a hierarchical stochastic block model; and 3) improving the strategies for weight modifiers.


\newpage

\subsubsection{ToDo}

\begin{itemize}
    \item Others 
    \begin{todolist}
        \item Check figures for being described properly, clear labelling, axis etc.
        \item Re-do the P0 graphs with the new plot
        \item Ensure that there is a logical narrative throughout the chapter
        \item Check for grammar, spelling
        \item Complement the Discussion
        \item Check if the introduction needs more addition
    \end{todolist}
\end{itemize}

\paragraph*{Things to do for paper}

\begin{itemize}
    \item Subtype exploration
    \begin{todolist}
        \item Further explore the unique pathways for the subtypes. A particular focus should be on the other subtypes: mes-like, lumInf (GU) and large luminal
        \item mes-like shares characteristics with the basal
        \item explore other values for ranking the most representative genes for a subtype
    \end{todolist}
    \item Advantages of the network
    \begin{todolist}
        \item Plotting the correlation distribution of these genes; before and after selective edge pruning
    \end{todolist}
\end{itemize}
