\section{A network approach to subtype MIBC}


\vspace{3mm}
\fbox{
    \parbox{0.95\linewidth}{
      \begin{itemize}
        \item Develop a network pipeline (inspired from PGCNA)
        \item First attempt to integrate mutations and Transcription Factors
        \item Use non-cancerous data (P0) to inform MIBC subtyping 
        \item Develop tools to analyse networks
      \end{itemize}
    }
}
\vspace{3mm}


%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\subsection{Overview}

This section needs to begin by addressing why a network approach, why the Gene expression is best represented by a network and the features that it enables us to build on that.
% 
The motivation behind using network/graph approaches is that they are closer to the biology of bladder cancer. More specifically, these allow us to represent pathways, and partial correlation between genes. Thus, a network approach is dealing both with the local and global effects of a gene. In addition, having links/connections between genes/nodes will enable to add weights (i.e importance) but also severe (in case of mutations that turns off a pathway).


\subsubsection{Datasets used}

The MIBC cohort from The Cancer Genome Atlas (TCGA) \cite{Robertson2017-mg} and the P0 samples from Jack Birch Unit (JBU) are used. The first dataset is used in the first part of the chapter where a network is created from the genes expressed in the tumours and then subtype the cohort. While in second part, the network is constructed from the genes expressed in the P0 samples which are then used to subgroup the MIBC cohort from TCGA. Both datasets consists of Transcriptions Per Million (TPM) from RNA sequencing (RNAseq). 

The mutation information is coming from Whole Exon Sequencing (WES) of the MIBC cohort from TCGA. In this chapter only the mutation count, i.e. how much a gene is mutated across the cohort. In the following chapters the mutation type (e.g. if it is missense) is used to integrate more details.

The list of Transcription Factors (TFs) is taken from The Human Transcription Factors work \citet{Lambert2018-el}.



% This needs to be in the literature review side of things
% \import{Sections/Network_I/}{review.tex}


\import{Sections/Network_I/}{methods.tex}

\newpage

\subsection{Experiments Overview}


In the initial experiments two networks were constructed, one from the healthy dataset and the other from the tumour dataset (TCGA). Only P0 samples from the healthy datasets are initially used to build the networks as these represents an unaltered version of the bladder tissue. Compared with the AbsCa differentiated tissues dataset, P0 contains the noise in the in-situ tissue.

On these networks weight modifiers were applied, reward and penalising the genes that are mutated, proportionally to the mutations across the cohort. The hypothesis is that by applying modifiers it will improve the community separation inside the networks and each community will be representative of a biological process. Then, the most representative genes are selected for each community using ModCon score. Those  genes are used to stratify the MIBC samples by clustering their MEV.

\import{Sections/Network_I/}{tum.tex}
\newpage
\import{Sections/Network_I/}{p0.tex}
\newpage



\newpage

\subsection{Discussion}


\subsection{Checkpoint}


Things left to do:
\begin{itemize}
    \item Experiments 
          \begin{todolist}
              \item Re-write the introduction to cover all the experiments, especially the two parts
          \end{todolist}
    \item P0 - experiments
          \begin{todolist}
              \item Put stats: How many of the genes selected w/ ModCon are TF. How does this compares to the initial ratio.
          \end{todolist}
    \item Tum - experiments
          \begin{todolist}
              \item To be completed
          \end{todolist}
    \item H derived experiments
          \begin{todolist}
              \item To be completed
          \end{todolist}
\end{itemize}
\vspace{1cm}
\textbf{Conceptual} regarding on network analysis. In essence, small questions that are trying to find the best network configuration:
\begin{todolist}
    \item Do we need more stats? How do the authors in PGCNA choose the number of edges per node? Can we apply it to our case?
    \item What defines a noisy network?
    \item How do we know if we lose some biological information?
\end{todolist}