% Introduction
\chapter{Introduction} \label{s:ap:intro}
% mibc paper images

% TCGA
\section{TCGA}

\begin{figure}[!htb]    
    \centering
\includegraphics[width=1.0\textwidth,keepaspectratio]{Sections/Lit_review/Resources/TCGA_2017_subtypes.jpg}
    \caption{Figure 2 from the work of \citet{Robertson2017-mg} showing the five different molecular subtypes found in the MIBC cohort from TCGA: \acrfull{lum}, \acrfull{luminf}, \acrfull{ba/sq}, \acrfull{ne} and \acrfull{lump}. The subgroups are presented in more detail in \cref{s:lit:tcga_mibc}. }
    \label{fig:ap:tcga_subtypes}
\end{figure}

% MIBC subtypes
\newpage
\section{MIBC tables subtypes}

\subsection{TCGA}

% TCGA table
\begin{table}[H]
    \centering
    \small
    \begin{tabularx}{\textwidth}{ >{\hsize=.5\hsize}X |>{\hsize=.8\hsize}X |>{\hsize=.8\hsize}X
    }
    \toprule
    Subtype & Genes & Description \\
    \midrule
    \acrlong{lump} (35\%) & \textit{KRT20, PPARG, FOXA1, GATA3, SNX31, UPK1A, UPK2, FGFR3} & 
    \begin{itemize}[leftmargin=*, nosep, after=\vspace{-\baselineskip}, before=\vspace{-.6\baselineskip}]
        \item Characterised by \textit{FGFR3} mutations
        \item Papillary histology
        \item Low risk of progression
    \end{itemize} \\
    \midrule
    \acrlong{luminf} (19\%) & \textit{CD274, PDCD1LG2, IDO1, CXCL11, L1CAM, SAA1} & 
    \begin{itemize}[leftmargin=*, nosep, after=\vspace{-\baselineskip}, before=\vspace{-.6\baselineskip}]
        \item Lowest purity
        \item High expression of \textit{EMT} and myofibroblast markers, and of the miR-200s (tumour suppressor)
        \item Medium expression of \textit{CD274, CTLA4} immune markers
        \item May respond to immune checkpoint therapy
    \end{itemize} \\
    \midrule
    \acrlong{lum}  (6\%) & \textit{KRT20, PPARG, FOXA1, GATA3, SNX31, UPK1A, UPK2, FGFR3} & 
    \begin{itemize}[leftmargin=*, nosep, after=\vspace{-\baselineskip}, before=\vspace{-.6\baselineskip}]
        \item New subtype in TCGA
    \end{itemize} \\
    \midrule
    \acrlong{ba/sq}  (35\%) &\textit{CD44, KRT6A, KRT5, KRT14, COL17A1};Squamous: \textit{DSC3, GSDMC, TCGM1, PI3, TP63} & 
    \begin{itemize}[leftmargin=*, nosep, after=\vspace{-\baselineskip}, before=\vspace{-.6\baselineskip}]
        \item High incidence in women
        \item Squamous differentiation, and basal keratin expression
        \item High expression of \textit{CD274, CTLA4} immune markers and may respond immune checkpoint therapy
    \end{itemize} \\
    \midrule
    \acrlong{ne-like} (5\%) &\textit{MSI1, PLEKHG4B, GNG4, PEG10, RND2, APLP1, SOX2, TUBB2B} & 
    \begin{itemize}[leftmargin=*, nosep, after=\vspace{-\baselineskip}, before=\vspace{-.6\baselineskip}]
        \item Expression of neuroendocrine, neuronal gene and high cell-cycle signature of proliferative state
        \item The most aggressive tumours with the lowest survival rate
    \end{itemize} \\
    \bottomrule
    \end{tabularx}
    \caption{The molecular characterisation of the TCGA subgroups taken from the \textit{mRNA Expression-Based Molecular Subtypes} and \textit{Discussion} Figure 2 from the work of \cite{Robertson2017-mg}. The TCGA's subtyping of the MIBC is covered in more depth in \cref{s:lit:tcga_mibc}.}
    \label{tab:lit:tcga_genes}
\end{table}

% Lund
\subsection{Lund}

\begin{table}[H]
    \centering
    \begin{tabularx}{\textwidth}{>{\hsize=.6\hsize}X |>{\hsize=\hsize}X}
    \toprule
    Subtype & Description \\
    \midrule
    UroA and GU & 
    \begin{itemize}[leftmargin=*, nosep, after=\vspace{-\baselineskip}]
        \item The UroA and GU tumours exhibit two signatures known in urothelial cell differentiation 1) \textit{PPARG, FOXA1, GATA3, ELF3} and 2) \textit{UPK1A, UPK1B, UPK2, UPK3A, KRT20}. These are the genes also seen in the Luminal subtypes form TCGA.
        \item The difference between GU and Uro subtypes is given by the up/down-regulated of \textit{FGFR3, CCND1, E2F3, RB1, CDKN2A}.
    \end{itemize} \\
    \midrule
    Basal/Squamous & 
    \begin{itemize}[leftmargin=*, nosep, after=\vspace{-\baselineskip}]
        \item The signatures of \textit{KRT5, KRT14, FOXA1 and GATA3} are definitory for the Basal/Squamous tumours (Ba/Sq). 
        \item Keratinization signature: \textit{KRT5, KRT6A, KRT6B, KRT6C, KRT14, KRT16}
        \item Cell adhesion genes \textit{EPCAM, CDH1, CDH3} differentiate the Basal tumours from the squamous.
        \item The Basal/Squamous are also separated from the Uro and GU subtypes by the difference in expression of the tyrosine kinase receptors \textit{EGFR, ERBB2, ERBB3}.
    \end{itemize} \\
    \midrule
    MES-like & 
    \begin{itemize}[leftmargin=*, nosep, after=\vspace{-\baselineskip}]
        \item Similar to Sc/NE-like, but are less infiltrated
    \end{itemize} \\
    \midrule
    Sc/NE-like & 
    \begin{itemize}[leftmargin=*, nosep, after=\vspace{-\baselineskip}]
        \item As in the case for Basal, high in \textit{KRT5, KRT14}, low in \textit{FOXA1, GATA3}
        \item High expression in\textit{ CHGA, SYP, ENO2}
        \item More infiltrated than the MES-like
    \end{itemize} \\
    \bottomrule
    \end{tabularx}
    \caption{Summary table of the work done for Lund stratification which is primarily based on the Figure 2 from \citet{Marzouka2018-ge} but it also contains some additional information from research group's earlier work from  \citet{Sjodahl2017-xr}. The suffix '-inf' represents the infiltrated nature of the subtype. The Lund taxonomy of the MIBC is covered in more depth in \cref{s:lit:lund_mibc}.}
    \label{tab:lit:lund_genes}
\end{table}


% Consensus
\subsection{Consensus}


\begin{table}[!htb]
    \small
    \centering
    \begin{tabularx}{\textwidth}{>{\hsize=.8\hsize}X |>{\hsize=\hsize}X}
    \toprule
    Subtype & Description \\
    \midrule
    \acrlong{lump} (24\%) & 
    \begin{itemize}[leftmargin=*, nosep, after=\vspace{-\baselineskip}]
        \item High expression of noninvasive Ta pathway signature
        \item Mutations in \textit{FGFR3, KDM6A}
    \end{itemize} \\
    \midrule
    \acrlong{lumns} (8\%) & 
    \begin{itemize}[leftmargin=*, nosep, after=\vspace{-\baselineskip}]
        \item Elevated stromal infiltration signatures
        \item \textit{ELF3} mutations, activated by \textit{PPARG//RARG}
    \end{itemize} \\
    \midrule
    \acrlong{lumu} (15\%) & 
    \begin{itemize}[leftmargin=*, nosep, after=\vspace{-\baselineskip}]
        \item Higher cell activity compared to other luminal tumours
        \item Mutations in \textit{ERCC2, TP53}
    \end{itemize} \\
    \midrule
    \acrlong{stroma} (15\%) & 
    \begin{itemize}[leftmargin=*, nosep, after=\vspace{-\baselineskip}]
        \item Intermediate levels of urothelial differentiation and stromal infiltration
        \item Overexpression of smooth muscle, endothelial, fibroblast and myofibroblast gene signatures
        \item Immune infiltration of T and B-cell
    \end{itemize} \\
    \midrule
    \acrlong{ba/sq} (35\%) & 
    \begin{itemize}[leftmargin=*, nosep, after=\vspace{-\baselineskip}]
        \item Mutations in \textit{KDM6A, RB1}
        \item Signatures associated with Basal
        \item Immune infiltration: cytotoxic lymphocytes, natural killer cells
    \end{itemize} \\
    \midrule
    \acrlong{ne} (3\%) & 
    \begin{itemize}[leftmargin=*, nosep, after=\vspace{-\baselineskip}]
        \item Mutations in \textit{KDM6A, RB1}
        \item Signatures associated with Basal
    \end{itemize} \\
    \bottomrule
    \end{tabularx}
    \caption{The molecular characterisation of the MIBC subgroups taken based on the consensus work of \cite{Kamoun2020-tj}. It summaries sections\  textit{3.2 and 3.3} from the main paper. LumP, LumNS and LumU exhibit more differentiation status characteristics and contain \textit{PARG/GATA3/FOXA1}-related Lund signature. The Consensus work on MIBC is covered in more depth in \cref{s:lit:consensus_mibc}.}
    \label{tab:lit:consensus_genes}
\end{table}

\newpage


% List of models

% list of models
\subsection{List of Models} \label{ap:tables_models}

\newpage

\begin{center}
\small
\begin{longtable}{|p{3cm}|c|p{1.2cm}|c|p{2.0cm}|p{4.0cm}|}
    \hline \textbf{Model} 
    & \multicolumn{1}{p{1.0cm}|}{\textbf{RNA-seq}} 
    & \multicolumn{1}{p{1.2cm}|}{\textbf{Micro-arrays}} 
    & \multicolumn{1}{p{1.0cm}|}{\textbf{Muta-tions}} 
    & \textbf{Other}
    & \textbf{Approach}  \\ \hline  \hline
    \endfirsthead
    \endlastfoot    
    \citet{Cava2018-rv} & x &  &  & Pathways & Network/Graph \\ \hline
    GANPA &  & x &  & Protein-protein & Network/Graph \\ \hline
    LEGO & x &  &  & Pathways & Network/Graph \\ \hline
    Dendrix &  &  & x &  & Greedy algorithm \& Monte Carlo (MCMC) \\ \hline
    MDPFinder & x &  & x &  & EA \& optimisation algorithms \\ \hline
    Neuro-evolution &  & x &  &  & FS-NEAT, feature selection via neuroevolution / supervised \\ \hline
    PICNIC  &  &  & x &  & Pipeline, 4 stages with a range of models \\ \hline
    DriverNet & x &  & x &  & Network/graph \\ \hline
    DawnRank & x &  & x & Gene network & Network/graph. Built on DriverNet \& using PageRank. \\ \hline
    Memo & x &  & x &  & Network/Graphs, Stats \\ \hline
    iPAC & x &  & x &  & Stats \& correlations \\ \hline
    Comet  & x &  & x &  & Submatrix, Markov Chain Monte Carlo  \\ \hline
    \citet{Feltes2019-bd} & x & x &  &  & Built on Neuroevolution \\ \hline
    \citet{Palazzo2019-hx} &  &  & x &  & Autoencoders, hierarchical clustering \\ \hline
    \citet{Chaudhary2018-qj} & x &  &  & miRNA, DNA methylation & Autoencoders \\ \hline
    \citet{Ma2019-hk} & x &  &  & miRNA, DNA methylation & Autoencoders \\ \hline
    TCGA clustering & x &  &  &  & Hierarchical clustering \\ \hline
    Consensus clustering & x &  &  &  & Hierarchical clustering \\ \hline
    \citet{Capecci2020-uj} &  & x & x &  & SNN and EA \\ \hline
    MutsigCV &  &  & x &  & Stats \\ \hline
    \caption{Computational models explored in this literature review with the type of data used and the approaches.}
    \label{tab:data_used}
\end{longtable}
\end{center}
    

    
{\footnotesize % Reduces the font size around the table
\begin{longtable}{|p{3cm}|p{1.8cm}|p{2.2cm}|p{2.8cm}|p{3.5cm}|}
\hline 
\textbf{Model} & \textbf{Data used} & \textbf{Datasets} & \textbf{Approach} & \textbf{Goal} \\ 
\hline 
\endfirsthead

\endlastfoot    
\citet{Cava2018-rv} & Mut & KEGG [5]; TCGA  & Network/Graph & Establish a connection between GE and functional pathways  \\ \hline
GANPA & GE  & GSEA; for PPI -HPRD, MINT, DIP, MINT, IntAct & Network/Graph & Breast cancer, asthma, p53 \\ \hline
LEGO & GE + pathways & KEGG \& others & Network/Graph & Autism and breast cancer \\ \hline
Dendrix & GE & TCGA, Thomas et al. & Greedy algorithm, Monte Carlo & Pan-cancer analysis \\ \hline
MDPFinder & GE + Mut & TCGA & EA, optimisation algorithm & Head \& Neck, glioblastoma and ovarian cancer \\ \hline
Neuro-evolution & GE & GEO & FS-NEAT, feature selection via neuroevolution / supervised & Leukaemia, breast, and colorectal cancers \\ \hline
PICNIC  & Mut & TCGA  & Pipeline, 4 stages with a range of models & Multiple cancer types  \\ \hline
DriverNet & GE + Mut & TCGA, METABRIC, TN and GBM & Network/graph &  Glioblastoma, breast, triple-negative breast, serous ovarian \\ \hline
DawnRank & GE + Mut & TCGA & Network/graph. Built on DriverNet, using PageRank & Breast, ovarian cancer \\ \hline
Memo & GE + Mut & TCGA & Network/ Graphs, Stats & Glioblastoma multiforme (GBM) and ovarian cancer \\ \hline
iPAC & GE + Mut & Multiple datasets & Stats, correlations & Breast \\ \hline
Comet  & GE + Mut & TCGA & Submatrix, Markov Chain Monte Carlo & breast \& gastric cancer, glioblastoma, myeloid leukaemia \\ \hline
\citet{Feltes2019-bd} & GE  & GEO / (Various datasets) & Built on Neuroevolution & Pan-cancer \\ \hline
\citet{Palazzo2019-hx} & Mut & ICGC & Autoencoders, hierarchical clustering & Tumor mutation profile analysis \\ \hline
\citet{Chaudhary2018-qj} & GE + others & TCGA & Autoencoders & Patient survival on liver cancer \\ \hline
\citet{Ma2019-hk} & GE & TCGA, STRING & Autoencoders &  Bladder and Brain Lower Grade Glioma \\ \hline
TCGA clustering & GE & TCGA & Hierarchical clustering & Bladder cancer  \\ \hline
Consensus clustering & GE & TCGA & Hierarchical clustering & Bladder cancer \\ \hline
\citet{Capecci2020-uj} & GE + Mut & TCGA & SNN and EA & Skin dermatitis  \\ \hline
\caption{Datasets: Kyoto Encyclopedia of Genes and Genomes (KEGG) \cite{Kanehisa2017-wj}, The Cancer Genome Atlas (TCGA) \cite{Tcga2018-sj}, \citet{Thomas2007-yj} - "High-throughput oncogene mutation profiling in human cancer", The Gene Expression Omnibus (GEO) \cite{Clough2016-zc, Davis2007-at}, International Cancer Genome Consortium (ICGC) \cite{International_Cancer_Genome_Consortium2010-ca}, Search Tool for Retrieval of Interacting Genes/Proteins(STRING) \cite{Szklarczyk2019-pu}.}
\label{tab:approaches}
\end{longtable}
}

% % Dendogram example
% \section{Dendrogram} \label{ap:dendogram}


% \begin{figure}[!htb]                                  
%     \centering\includegraphics[width=1.0\textwidth,height=1.0\textheight,keepaspectratio]{Images/Clustering/dendogram.png}
%       \caption{Dendogram specific to hierarchical clustering and it can be seen how the algorithm starts classifying each datapoint in its own cluster followed by a merge in higher up clusters. For this we have used the TCGA dataset for bladder cancer, using hierarchical clsutering with Ward linkage.}
%       \label{fig:dendogram}
%   \end{figure}
%   \FloatBarrier





