\subsection{Co-expressed networks} \label{s:lit:co_net}

Co-expressed networks are built from gene expression data (RNAseq or Microarray) where the edges' strength represents the correlation between the two. 
% Weighted Gene Correlation Network Analysis (WGCNA) by \citet{Langfelder2008-sn} was one of the first and more popular of a such approach. 

% Challenges in the neworks - or assumptions - decisions
% Create the network - the use of correlation - what are the advantages?
% Reduce the edges - use a threshold 


% WGCNA
One of the most popular gene co-expressed network (GCN) is the Weighted Gene Correlation Network Analysis (WGCNA) by \citet{Langfelder2008-sn}. This paper is an accumulation of different works from Dr Steve Horvath lab which offers a clear and concise methodology to build and analyse a GCN, and more importantly everything is packed in an R package which then other researchers can use. Being a referential work in the field, it follows the steps described above on creating, processing and analysing the graph. Compared to PGCNA (presented below) where correlation values are unaltered, in WGCNA the values are scaled from 0-1, which at the cost of loosing some information resolution it helps the graph processing. To reduce the number of edges \citet{Langfelder2008-sn} sets a threshold to the correlation values and for community detection they apply topological overlap which was described in\citet{Zhang2005-xq} and applied to \citet{Yip2007-mr, Li2007-vz, Ravasz2002-au}. The discovered modules are then clustered using hierarchical clustering.

% Partial-correlation
Partial-correlation is also used by \citet{De_la_Fuente2004-ts}


% TF - Elati's work


% HCNM - and integrating data into networks
\subsection{Data integration in networks}

\subsubsection{HCNM}

One way to integrate multiple data-types is to build a separate network for each data type and then connect them through an intra-layer, thus giving rise to a multi-layer network. This is the approach taken by \citet{Vangimalla2021-fc} in the Heterogeneous Correlation Network Model (HCNM) to subtype the breast carcinoma cohort from TCGA using gene expression and DNA methylation. Compared to the WGCNA and PGCNA, the HCNM constructs the gene expression layer with partial-correlation, and the DNA methylation using biweight mid-correlation, a metric popularised by WGCNA. 

Once the network layers are constructed the authors apply an edge reduction technique by filtering out low weighted connections. Then, to find the biological related genes, for each layer, the consensus of three different community detection algorithms (Louivan\cite{Blondel2008-ik}, Walktrap method\footnote{} and Fast Greedy Optimisation Model\footnote{}) is taken. Together with the network metrics (degree, centrality and betweenness) in both layers, the genes for the intra-layer network are established.

With the intra-layer network constructed the authors again perform the steps of finding the important nodes and once this is done they apply something called similarity network fusion (SNF) and affinity network fusion (ANF) to find the representation of the genes in the samples\footnote{This step is analogous to the MEV from PGCNA}. 

From the resultant networks the sample subtypes are derived which are comparable with the subtypes known in the field and exhibit more informative breast cancer subgroups compared with iCluster. The assessment is mainly done by using gene ontology and Kaplan-Meier survival plot, and comparing HCNM with different configurations and iClusterPlus \citet{Mo2013-zi}. 

HCNM represent a complex way on representing and integrating multi-omics data to find different subtypes. The added complexity is mainly given by treating the two data types as separate networks and finding a method to connect them. Unfortunately, the paper lacks a direct comparison with the TCGA cohort and a biological analysis of subtypes, letting the reader wonder about the biological significance of all this hard-work put in developing HCNM. On top of that, there is no software package published, making it very hard to assess the model. 

\subsubsection{iHNMMO}

% introduce iHNMMO - maybe I'll need to get further in details
HCNM was inspired from the work of \citet{Peng2017-ik} where the authors proposed a multi-layer network to analyse the muscle invasive bladder cancer cohort from TCGA. Compared to HCNM, the integrative Heterogeneous Network Modeling of Multi-Omics (iHNMMO) is developed to help the understanding of the disease. The network is created from gene expression (Pearson correlation) but the other data types such as copy number variation (CNV), DNA methylation, miRNA and protein-protein interaction are used to model the connections. The rational behind is that the gene expression is affected by the other data types. However, it is curios how the mutation data was not included, which has a more direct impact on the gene expression.

The entire iHNMMO pipeline is used to analyse multiple data types and select the genes that are most 'relevant' for bladder cancer. The networks derived are not used to find communities from which the subtypes are derived as in HCNM\cite{Vangimalla2021-fc, Care2019-ij}. The pathways found (through gene ontology) does not reveal new leads for biology (see Table 3 from their paper\footnote{\url{https://www.nature.com/articles/s41598-017-15890-9/tables/3}}). On top of that, their software is not available.

% NetICS
\subsubsection{NetICS}

NetICS is a network approach that models the gene interaction with the goal to find biologically relevant genes, using multiple data types: protein, expression, mutation, miRNA (epigenetic) and copy number variation. The authors aggregates different interaction networks available such as Kegg\cite{Kanehisa2017-wj}, Panther \cite{Thomas2022-kn}, Signor \cite{Perfetto2016-tw} and others (Signalink\cite{Fazekas2013-qh} and \citet{Wu2010-ap}) to create the initial networks. On which based on the mutation and miRNA an aberration score is calculated and applied to the network via a difusion (\cite{Leiserson2015-kv}). It is worth pointing out that this is done only for differentially expressed genes between tumour and healthy. This gives a separate network for each sample, from which the highly ranked genes are selected, computed from differentially expressed score and ‘abnormal score’. Lastly, there is another selection at the cohort level to find the disease specific genes.

The authors tested their model on five TCGA cohorts: uterine corpus endometrial carcinoma, liver hepatocellular carcinoma, bladder urothelial carcinoma, breast invasive carcinoma and lung squamous cell carcinoma. Strangely, information about the normal data is not provided and it is mentioned that only the breast cancer cohort was paired with healthy information, but it fails to mention the source of the data. In the source repository of the data, the gene and protein differentially expressed files are optional and the directed functional network is not automatically computed by the programme but it is uses \citet{Wu2010-ap}.

Overall, NetICS is available to the general public and it promises good results. However, there is unclear how successful from a biological stance is and how easy to use given that the user needs to generate its own network via \citet{Wu2010-ap}. Also, it is a ranking mechanism of the genes, similarly to iHNMMO and not a subtyping pipeline.



\subsubsection{DrDimot}

DrDimont (Drug response prediction from Differential analysis of multi-omics networks)\citet{Hiort2022-lk} is another multi-layer approach used in biological networks. As the name suggested the model predicts the drug response based on combining data from gene expression, proteins, phosposite and metabolimics. For each available data type a separated layer from correlation scores and the edges are reduced by a mechanism similar as in WGCNA. The layers are simply connected through their node names. DrDimont was used to explore the different (known) drug responses to the breast cancer from TCGA.

This shows another application of the network, including WGCNA, to drug discovery. This is a work with strong biological motivation, but the model developed does not incorporate the mutations or Transcription Factors and it does not use any community detection algorithm to find the structures across the layers.


\subsection{PGCNA}

Parsimonious Gene Correlation Network Analysis (PGCNA) by \citet{Care2019-ij}, is a more recent  approach developed with the goal to stratify a disease based on gene expression and aid the researchers to visualise with the genomic data exploration

The authors compares their work with WGCNA and draw inspiration from other techniques that are mainly re-constructing the regulatory network of tissues from microarrays and using Mutual Information (MI) \citet{Margolin2006-mc,Zhang2013-fs} to calculate gene pair-wise relationship. Compared to correlation scores MI is a more general method to describe the dependence between two variables by using information theory concepts, but it is also harder to interpret.

Over the previous methods, the PGCNA supports the integration of multiple gene expression datasets by taking the median expression of a gene across the data. It is also been released in a GitHub package\footnote{\url{https://github.com/medmaca/PGCNA} } being relatively easy to use and to adapt to one's needs. PGCNA was also successfully applied to the Breast and Glioblastoma TCGA cohorts and it supports multiple datasets by using the median gene expression.

\paragraph{Measuring success}

To asses the performance of PGCNA over techniques \citet{Care2019-ij} proposed the scaled cluster enrichment score (SCES). The metric is computed for each module (i.e. a set of gene grouped together - regardless of the method, network or clustering analysis) taking in account the enrichment scores (obtained from gene ontology) and selects the 15 most significant\footnote{Only the enriched pathways that have False-Discovery Rate (FDR) $<0.05$ and containing genes $\geq5$ between $\leq1000$}. Then, the using z-scores the specific enrichment are determined for each module. Also, everything is adjusted by the number of genes in a module. Therefore, SCES is a metric the rewards the modules with unique enrichment signals and that are not large or ‘rewarding purity'\cite{Care2019-ij}.

It is worth mentioning, that gene ontology results are highly dependent on the list of genes given without receiving any other information such as fold change, which is used for gene set enrichment analysis (GSEA). Also, the the interval of genes contained is fairly large, being fairly permissive with the number of accepted enrichment.

With all the limitations of gene ontology, SCES is a metric to asses the significance of the biology of each module which usually more than there is available for computational models.

\paragraph{Edge reduction}

One important finding in the work of \citet{Care2019-ij} is that a simple and aggressive edge pruning strategy performed better then other strategies. These were iPCC (iterative Pearson’s correlation coefficient), PowerST and Sigmoid, the last two being part of the WGCNA pipeline.

The authors explores keeping edges from 3-10 and all of them, and found that less than 3, yielded isolated communities (or disconnected). Also, increasing the numbers yield a large number of communities find and a decrease in the SCES metric (modularity score too).

\paragraph{Community detection}

To detect the structure after the correlation matrix was reduced the Care et al explored 3 different classes of algorithms: K-means, hierarchical clustering and community detection (Louivan\citet{Blondel2008-ik} or FastUnfold and Leiden\citet{Traag2019-ne}\footnote{Leiden algorithm is an improvement to the Louivan which was released after PGCNA was published, but in the Github package, the Leiden is used as it outperforms Louivan.}). The latter outperformed the other two classes, with Louivan applied to a network constructed with 3 edges per gene, having the highest SCES. What is crucial in this comparison is that the traditional clustering methods, K-means and hierarchical clustering, are outperformed by the network approach.

\paragraph{Bridging the gap between genes and sample}

The end goal of PGCNA is to use the network representation to inform disease subtyping, thus there was a need to bridge the gap between the gene representation to samples. This is done by using the module connection values (ModCon) which from each community selects the top 25 most relevant genes. Then, the Module Evaluation Value (MEV) takes the 25 genes and computes the enrichment in the dataset. Lastly, these scores are then used for subtyping.

\paragraph{Summary on PGCNA}

Overall, PGCNA represents a bespoke pipeline for disease subtyping which introduces several key methods to the field. It shows that network and community detection yields the best results, a simple edge pruning performs better than the other techniques and methods to bridge the gap between the gene representation to the samples.

While PGCNA solves some of the challenges that this project faced there are still many left untouched. What is the best way to integrate the effect of the mutations into the network? Would selective edge pruning offer a better representation of the data? And so on. Thus, the work in this project builds on PGCNA and it offers the support for multiple data types integration.

\subsection{Building a network}

\subsection{Community detection}


