\subsection{Co-expressed networks} \label{s:lit:co_net}

Co-expressed networks are built from gene expression data (RNAseq or Microarray) where the edges' strength represents the correlation between the two. 
% Weighted Gene Correlation Network Analysis (WGCNA) by \citet{Langfelder2008-sn} was one of the first and more popular of a such approach. 

% Challenges in the neworks - or assumptions - decisions
% Create the network - the use of correlation - what are the advantages?
% Reduce the edges - use a threshold 


% WGCNA
One of the most popular gene co-expressed network (GCN) is the Weighted Gene Correlation Network Analysis (WGCNA) by \citet{Langfelder2008-sn}. This paper is an accumulation of different works from Dr Steve Horvath lab which offers a clear and concise methodology to build and analyse a GCN, and more importantly everything is packed in an R package which then other researchers can use. Being a referential work in the field, it follows the steps described above on creating, processing and analysing the graph. Compared to PGCNA (presented below) where correlation values are unaltered, in WGCNA the values are scaled from 0-1, which at the cost of loosing some information resolution it helps the graph processing. To reduce the number of edges \citet{Langfelder2008-sn} sets a threshold to the correlation values and for community detection they apply topological overlap which was described in\citet{Zhang2005-xq} and applied to \citet{Yip2007-mr, Li2007-vz, Ravasz2002-au}. The discovered modules are then clustered using hierarchical clustering.

% PGCNA

% Partial-correlation

% TF - Elati's work

\subsection{PGCNA}

A more recent network approach was introduced by \citet{Care2019-ij} with the Parsimonious Gene Correlation Network Analysis (PGCNA).

The authors compares their mainly with WGCNA as is a popular network framework but also with other techniques that are mainly constructing the regulatory network of tissues from microarrays and using Mutual Information (MI) \citet{Margolin2006-mc,Zhang2013-fs} to calculate gene pair-wise relationship. Compared to correlation scores MI is a more general method to describe the dependence between two variables by using information theory concepts, but it is also harder to interpret.

% Talk about the datasets that was applied on and that it supports multiple datasets 

The pipeline is applied to the Breast and Glioblastoma TCGA cohorts and it supports multiple datasets by using the median gene expression.

\paragraph{Measuring success}

To asses the performance of PGCNA over techniques \citet{Care2019-ij} proposed the scaled cluster enrichment score (SCES). The metric is computed for each module (i.e. a set of gene grouped together - regardless of the method, network or clustering analysis) taking in account the enrichment scores (obtained from gene ontology) and selects the 15 most significant\footnote{Only the enriched pathways that have False-Discovery Rate (FDR) $<0.05$ and containing genes $\geq5$ between $\leq1000$}. Then, the using z-scores the specific enrichment are determined for each module. Also, everything is adjusted by the number of genes in a module. Therefore, SCES is a metric the rewards the modules with unique enrichment signals and that are not large or ‘rewarding purity' (\citet{Care2019-ij}).

It is worth mentioning, that gene ontology results are highly dependent on the list of genes given without receiving any other information such as fold change, which is used for gene set enrichment analysis (GSEA). Also, the the interval of genes contained is fairly large, being fairly permissive with the number of accepted enrichment.

With all the limitations of gene ontology, SCES is a metric to asses the significance of the biology of each module which usually more than there is available for computational models.

\paragraph{Edge reduction}

One important finding in the work of \citet{Care2019-ij} is that a simple and aggressive edge pruning strategy performed better then other strategies. These were iPCC (iterative Pearson’s correlation coefficient), PowerST and Sigmoid, the last two being part of the WGCNA pipeline.

The authors explores keeping edges from 3-10 and all of them, and found that less than 3, yielded isolated communities (or disconnected). Also, increasing the numbers yield a large number of communities find and a decrease in the SCES metric (modularity score too).

\paragraph{Community detection}

To detect the structure after the correlation matrix was reduced the Care et al explored 3 different classes of algorithms: K-means, hierarchical clustering and community detection (Louivan\citet{Blondel2008-ik} or FastUnfold and Leiden\citet{Traag2019-ne}\footnote{Leiden algorithm is an improvement to the Louivan which was released after PGCNA was published, but in the Github package, the Leiden is used as it outperforms Louivan.}). The latter outperformed the other two classes, with Louivan applied to a network constructed with 3 edges per gene, having the highest SCES. What is crucial in this comparison is that the traditional clustering methods, K-means and hierarchical clustering, are outperformed by the network approach.

\paragraph{Bridging the gap between genes and sample}

The end goal of PGCNA is to use the network representation to inform disease subtyping, thus there was a need to bridge the gap between the gene representation to samples. This is done by using the module connection values (ModCon) which from each community selects the top 25 most relevant genes. Then, the Module Evaluation Value (MEV) takes the 25 genes and computes the enrichment in the dataset. Lastly, these scores are then used for subtyping.

\paragraph{Summary on PGCNA}

Overall, PGCNA represents a bespoke pipeline for disease subtyping which introduces several key methods to the field. It shows that network and community detection yields the best results, a simple edge pruning performs better than the other techniques and methods to bridge the gap between the gene representation to the samples.

\subsubsection{Building a network}

\subsubsection{Community detection}


