

\section{Overview of the literature review}


% What are the questions that I need to answer - so that's where I need to look over
The aim of this PhD project is to advanced the current methods for bladder cancer subtyping by using multiple data types with the goal to close the gap between the tumour subgroups and clinical translation. This involves building an understanding of different (overlapping) fields such as networks and multi-omics integration, clustering analysis and so on. Some of the problems that this project tackles are \textbf{clustering analysis}, how are the tumour subgroups found?, \textbf{data representation}, what is a biological accurate (or close) to represent the molecular biological data so that different layers of complexity can be latter added?, \textbf{data reduction}, how can the signal be isolated from noise? and so on.

To attempt answering these questions it requires a wide-range view of the fields involved, that's why the Literature Review is structured in three different chapters. The aim of the first chapter is to given an overview of the different computational methods considered for this project covering clustering analysis (\ref{s:clustering}), artificial neural networks (\ref{s:ann_overview}), evolutionary algorithms (\ref{s:ea_overiew}) and a light touch on Graph Theory (\ref{s:graph_overview}) as this is extensively covered later on.

Analysing omics data (\ref{s:multi-omics}) focuses on the research done to analyse molecular data covering the work with gene expression (\ref{}) and mutations (\ref{}). It also includes integrative approaches using networks in (\ref{}) and iCluster (\ref{}) as well as the attempts of using Deep Learning in genomics (\ref{}). 

Covering the available methods to analyse molecular data (\ref{} and other Machine Learning approaches (\ref{}), the literature review allures to the issues of these methods. The third chapter focuses on Networks in molecular biology (\ref{}) and extensively presents the different models to determine sample subgroups using networks. It covers the work on co-expressed networks focusing on the most popular model, WGCNA \citet{Langfelder2008-sn}, and the improvement of this through PGCNA\citet{Care2019-ij}, then Bayesian methods and data integration with networks are presented. Lastly, community detection methods are covered extensively as finding groups in networks/graphs are crucial for this PhD project.





\import{Sections/Lit_review/Yr1/}{master.tex}

\import{Sections/Lit_review/}{network.tex}

% Just for the notes
\import{Sections/Lit_review/}{data_integration.tex}

\import{Sections/Lit_review/Notes/}{network_notes.tex}


% \subsubsection{Feature selection - iCluster}
% \subsubsection{Matrix factorisation}
