

\section{Introduction}


% What are the questions that I need to answer - so that's where I need to look over
The aim of this PhD project is to advance the current methods for bladder cancer subtyping by using multiple data types with the goal to close the gap between the tumour subgroups and clinical translation. This involves building an understanding of multiple (and overlapping) fields such as bladder cancer, clustering analysis, networks an multi-omics integration. Some of the problems that this project tackles are \textbf{clustering analysis}, how are the tumour subgroups found? \textbf{data representation}, what is a biological accurate (or close) representation of the molecular biology that enables integrating different layers of information? \textbf{data reduction}, how can the signal be isolated from noise? and so on. 

To attempt answering these questions it requires a wide-range view of the fields involved, that's why the Introduction is structured in five different sections, starting with an introduction (\cref{s:lit:bladder_cancer}) to cancer and the research done to understand the bladder cancer with a focus on the more aggressive type. This section covers the motivation of the project, why there is a need to integrate multiple data-types, why bladder cancer is difficult disease to treat and understand. It also covers the standard methods used to stratify the muscle-invasive bladder cancer (MIBC).

\Cref{s:lit:multi-omics} extends the scope outside of the bladder cancer, and it focuses on how different data types are analysed (gene expression, mutations and others). It reviews the methods for gene expression and  (\ref{s:lit:rnaSeq}) and mutations (\ref{s:lit:mutations}). It includes integrative approaches using networks (\ref{s:lit:multi-view}) and iCluster (\ref{s:lit:iCluster}) as well as the attempts of using Deep Learning in genomics (\ref{s:lit:dl_genomics}). 

This is followed by an overview of the different computational methods considered and used in this project covering clustering analysis (\ref{s:lit:clustering}), artificial neural networks (\ref{s:lit:ann_overview}) and evolutionary algorithms (\ref{s:lit:ea_overview}) .Covering the available methods to analyse molecular data (\ref{s:lit:multi-omics}) and other Machine Learning approaches (\ref{s:lit:computational}), the literature review allures to the issues of these methods.

While 2nd (\cref{s:lit:multi-omics}) and 3rd sections (\ref{s:lit:computational})  of the Introduction focused on reviewing the methods from a biological angle, first, and then on 'unbiased' computational approach, the last two sections are focused on the main method used in this PhD thesis: networks. Their applications and types are presented in \ref{s:lit:nets_bio} and it extensively presents the different models to determine sample subgroups using networks. It covers the work on co-expressed networks focusing on the most popular model, WGCNA \cite{Langfelder2008-sn}, and the improvement of this through PGCNA \cite{Care2019-ij}, then Bayesian methods and data integration with networks are presented. 

Community detection algorithms are covered in the last section (\ref{s:lit:comm_detect}) as these represents a crucial part of the project. The focus is on the Leiden, Louivan and the Stochastic Block Model, representing the two main school of thoughts in this field. 

\import{Sections/Lit_review/}{bladder_cancer.tex}

\pagebreak

\import{Sections/Lit_review/Yr1/}{master.tex}

\import{Sections/Lit_review/}{network.tex}

% Just for the notes
\import{Sections/Lit_review/}{data_integration.tex}

% \import{Sections/Lit_review/Notes/}{network_notes.tex}

