\section{Networks in molecular biology} \label{s:lit:nets_bio}

\vspace{3mm}
% \noindent\rule{17cm}{0.2pt}
\fbox {
    \parbox{\linewidth}{
      \begin{itemize}
        \item Co-expressed networks
        \item Integrative Network approaches
        \item Other network approaches
      \end{itemize}
    }
}
\vspace{3mm}

% Make the case for pathways and gene interactions - Why it is essential to study them
Interactions and the relationships between multiple components are central to tissue functioning and biological processes. There are multiple levels of interactions in molecular biology, which are represented by gene pathways where one gene influences another to produce proteins, and that, in turn, produces more proteins until that pathway fulfils its function. When specific components of cellular mechanisms fail, such as a mutation in a gene that plays a critical role in key pathways, the immediate impact on the tissue may not be apparent. Over time, however, the effects of the mutation can accumulate, influencing other mechanisms and potentially leading to a cascade of biological disruptions. This interconnectivity between genes and cellular processes underpins the widespread use of networks in genomics, facilitating rapid advancements in the field.

% Need to address the network pathways 
The relevant network concepts were introduced in \cref{s:lit:graph_overview}, and how graphs were used to analyse gene pathways is covered in \cref{s:lit:multi-view}. However, the information about pathways is still being researched. Although a complete representation of a tissue is not available, other alternatives exist to analyse the gene interactions in a tissue.

% Why are we choosing the co-expressed networks?
One approach is the co-expressed networks, which uses different correlation metrics (e.g. Spearman, Pearson or partial-correlation) from gene expression to model the strengths between the nodes (edges). The rationale is that genes expressed together (or co-expressed) have a higher correlation and subsequently have a stronger connection between nodes. Conversely, genes with a weaker connection are represented as lower correlated. After the network is built, related genes are grouped in communities and then used to subtype the disease. These network approaches are covered in \cref{s:lit:co_net} and are crucial for understanding the methods developed in this PhD project. The co-expressed networks are the preferred approach in this project as these are simple, versatile and meet the aims and objectives of this project outlined in \cref{s:intro:aims}. It also offers a more convenient and adaptable method to stratify the diseases on multiple sources of omics.

% DEA as an alternative
Another alternative to co-expressed networks is to build the network based on the \acrfull{dea} data between the healthy tissue and tumour. While slightly more restrictive, this approach highlights the differences between the two tissue states and potentially provides new biological insights. The review from \citet{Van_Dam2018-id} covers such network approaches and also acts as a guide on building DEA-based networks. Using DEA between healthy tissue and tumours has been considered in the project, but the co-expressed network was preferred for its versatility and simplicity.

% Alternative methods for co-expressed networks - in the review from Petti
The review from \citet{Petti2023-qo} covers the different network methods used to stratify a disease, some of which are explored in this project. The patient similarity network section presents different methods using co-expression networks but not the work of \citet{Care2019-ij} on PGCNA (2019), one of this project's primary sources of inspiration. Nevertheless, the review offers good insights on alternatives such as the network-based stratification (NBS), initially proposed for integrating data by \citet{Hofree2013-ld}, covered in \cref{s:lit:net_prop} with multi-layer networks.

This part of the Background chapter covers the work on co-expressed networks with a focus on PGCNA and WGCNA, as well as research done using partial correlation as an alternative to the standard methods (Spearman, Pearson) used to build a graph. The networks constructed using a Bayesian approach is a topic explored in the next section. \Cref{s:lit:net_data_int} provides an overview of network propagation and multi-layer graphs as methods to integrate multiple data types.

\import{Sections/Lit_review/}{co-expressed_net.tex}

\import{Sections/Lit_review/}{comm_detection.tex}





