\subsection{Networks in molecular biology} \label{s:lit:nets_bio}

\vspace{3mm}
% \noindent\rule{17cm}{0.2pt}
\fbox {
    \parbox{\linewidth}{
      \begin{itemize}
        \item Co-expressed networks
        \item Integrative Network approaches
        \item Other network approaches
      \end{itemize}
    }
}
\vspace{3mm}

% Make the case for pathways and gene interactions - Why it is important to study them
Interactions and the relationships between multiple components are central to tissues functioning and biological processes. There are multiple levels of interactions in the molecular biology which are represented by gene pathways, where one gene influences another to produce more or less of a protein, and that in turn produces more proteins and so on until that pathway fulfil its function. When something goes wrong in the body as in the case of the cancer, a certain pathways is altered and it is not working as intended. Thus, it is understandable why the field of using networks to analyse genomic data is saw rapid and exciting advancements in the recent years. 

% Need to address the network pathways 
The relevant network concepts were introduced in \cref{s:lit:graph_overview}
and how graphs were used to analysed gene pathways is covered in \cref{s:lit:multi-view}. However, the information about pathways is generally incomplete, in continuous research and a complete representation of a tissue is simply not available for most of the tissues. Therefore, there are other alternatives to analyse the gene interactions in a tissue.

% Why are we choosing the co-expressed networks?
One such approach are the co-expressed networks which uses different correlation metrics (e.g. Spearman, Pearson or partial-correlation) from gene expression to model the strengths between the nodes (edges). The rationale behind is that genes expressed together have a higher correlation will also have a stronger connection between nodes,conversely, genes with a weaker connection are represented are lower correlated. After the network is built related genes are grouped together in communities which are then used to subtype the disease. This approaches are covered in Section \ref{s:lit:co_net} and it is the method used in this project as it offers a more convenient and adaptable method to other diseases or other data (multiple sources of gene expression).

% DEA as an alternative
Another alternative to co-expressed network is to build the network based on the Deferentially Expressed Analysis (DEA) data between the healthy and tumour. While being a more restrictive approach this approach will highlight the differences between the two tissue states and might provide new biological insights. The review \citet{Van_Dam2018-id} covers such network approaches and also acts as guide on building DEA-based networks. Using DEA between healthy and tumours was considered in the project but the co-expressed network was preferred for its versatility.

% Alternative methods for co-expressed networks - in the review from Petti
The review from 2023 of \citet{Petti2023-qo} is covering the different network methods used in the field to stratify a disease a few which are explored in this project. The patient similarity network section covers different methods using co-expression networks but not the work of \citet{Care2019-ij} on PGCNA (2019) which is one of the main sources of inspiration in this project. Nevertheless, the review offers good insights on the other alternatives such as the network-based stratification (NBS), initially proposed for integrating data by \citet{Hofree2013-ld} and it is covered in \cref{s:lit:net_prop}, or the multi-layer (\cref{s:lit:multi-layer}) networks.

This section covers the work on co-expressed networks with a focus on PGCNA and WGCNA, as well as some research done using partial-correlation as an alternative to the standard methods (Spearman, Pearson) to build a graph. As an alternative to this approach the networks constructed using a Bayesian approach is covered in the next section. \Cref{s:lit:net_data_int} covers network propagation and multi-layer graphs as methods to integrate multiple data types. Lastly, the \cref{s:lit:comm_detect} is dedicated to community detection algorithms as they have a very important role in finding the groups of genes with a biological function.

\import{Sections/Lit_review/}{co-expressed_net.tex}

\import{Sections/Lit_review/}{comm_detection.tex}





