\makeglossaries


% Names
\newacronym{jbu}{JBU}{Jack Birch Unit}
\newacronym{tcga}{TCGA}{The Cancer Genome Atlas Program}
\newacronym{nhu}{NHU}{Normal Human Urothelium}
\newacronym{ic}{IC}{Interstitial Cystitis}

\newacronym{oab}{OAB}{overactive bladder}
\newacronym{nvh}{NVH}{non-visible haematuria}
\newacronym{uti}{UTI}{urinary tract infections}
\newacronym{nsUI}{nsUI}{non-specified urinary incontinence}


\newacronym{ruti}{rUTI}{recurrent urinary tract infections}
\newacronym{cc}{CC}{cystitis cystica}
\newacronym{sui}{SUI}{stress urinary incontinence}
\newacronym{tvt}{TVT}{tension-free vaginal tape}
\newacronym{ifn}{IFN-$\gamma$}{Interferon-$\gamma$}
\newacronym{emt}{EMT}{Epithelial-Mesenchymal}
\newacronym{absCa}{ABS-Ca}{Adult Bovine Serum}

\newacronym{lncRNA}{lncRNA}{long non-coding RNAs}

\newacronym{turbt}{TURBT}{transurethral resection of bladder tumour}

% Disease naming
\newacronym{mibc}{MIBC}{Muscle Invasive Bladder Cancer}
\newacronym{nmibc}{NMIBC}{Non-muscle Invasive Bladder Cancer}
\newacronym{scc}{SCC}{Squamous Cell Carcinoma}
\newacronym{uc}{UC}{Urothelial Carcinoma}
\newacronym{tcc}{TCC}{Transitional cell carcinoma}



% MIBC classifications
%%% Consensus/TCGA
\newacronym{luminf}{LumInf}{Luminal Infiltrated}
\newacronym{lumu}{LumU}{Luminal Unstable}
\newacronym{lumns}{LumNS}{Luminal Non-specified}
\newacronym{lump}{LumP}{Luminal Papillary}
\newacronym{lum}{Lum}{Luminal}

\newacronym{ne}{Ne}{Neuronal}
\newacronym{ne-like}{Ne-like}{Neuroendocrine}


\newacronym{ba/sq}{Ba/Sq}{Basal/Squamous}
\newacronym{ba/sq-inf}{Ba/Sq}{Basal/Squamous Infiltrated}

\newacronym{stroma}{Stroma-rich}{Stroma-rich}

%%% Lund
\newacronym{mes}{Mes-like}{Mesenchymal-like}
\newacronym{gu}{GU}{Genomically Unstable}
\newacronym{gu-inf}{GU-inf}{Genomically Unstable Infiltrated}
\newacronym{sc/ne}{Sc/Ne}{Small-cell/neuroendocrine}


% Non-tumour
\newacronym{p0}{P0}{freshly isolated cells}
\newacronym{ud}{UD}{undifferentiated}
\newacronym{abs}{ABS-Ca}{ABS-Ca differentiated}
\newacronym{tf}{TF}{Transcription factors}

% Models
\newacronym{sbm}{SBM}{Stochastic Block Model}
\newacronym{dc-sbm}{dc-SBM}{degree-corrected Stochastic Block Model}

\newacronym{hsbm}{hSBM}{Hierarchical Stochastic Block Model}
\newacronym{mdl}{MDL}{Minimum Description Length}

\newacronym{pgcna}{PGCNA}{Parsimonious Gene Correlation Network Analysis}
\newacronym{wgcna}{WGCNA}{Weighted Gene Correlation Network Analysis}
\newacronym{gcna}{GCNA}{Gene Co-expressed Network}
\newacronym{pca}{PCA}{Principal Component Analysis}


\newacronym{nmf}{NMF}{Non-matrix factorisation}
\newacronym{umap}{UMAP}{Uniform Manifold Approximation and Projection}
\newacronym{tsne}{t-SNE}{t-Distributed Stochastic Neighbour Embedding}

\newacronym{mev}{MEV}{Module Evaluation Value}
\newacronym{mod}{ModCon}{Module Connection}

% statistical tests
\newacronym{mw}{MW}{Mann-Whitney U-Test}
\newacronym{kw}{Kruskal}{Kruskal-Wallis H-test}


% tools
\newacronym{gsea}{GSEA}{Gene Set Enrichment Analysis}
\newacronym{dea}{DEA}{Differential Expression Analysis}
\newacronym{estimate}{ESTIMATE}{}

\newacronym{ivi}{IVI}{Integrated Value of Influence}
\newacronym{iqr}{IQR}{Interquartile Range}
% generic
\newacronym{ge}{GE}{Gene Expression}

\newacronym{ea}{EA}{Evolutionary Algorithms}
\newacronym{es}{ES}{Evolutionary Strategies}
\newacronym{ga}{GA}{Genetic Algorithms}
\newacronym{cgp}{CGP}{Cartesian Genetic Programming}

\newacronym{dna}{DNA}{Deoxyribonucleic acid}
\newacronym{rna}{RNA}{Ribonucleic acid}
\newacronym{ram}{RAM}{Random Access Memory}

\newacronym{wgs}{WGS}{Whole Genome Sequencing}
\newacronym{wes}{WES}{Whole Exon Sequencing}
\newacronym{ihc}{IHC}{Immunohistochemical}



% ML
\newacronym{snn}{SNN}{Spiking Neural Networks}
\newacronym{ann}{ANN}{Artificial Neural Networks}
\newacronym{dnn}{DNN}{Deep Neural Networks}
\newacronym{dl}{DL}{Deep Learning}

\newacronym{nn}{NN}{Neural Networks}
\newacronym{ai}{AI}{Artificial Intelligence}
\newacronym{ml}{ML}{Machine Learning}




%%%%%%% Glosssary %%%%%%%

\newglossaryentry{MIBC}
{
    name=Muscle Invasive Bladder Cancer,
    description={One of the two major types of bladder cancer, it is more aggressive and it is metastatic. At first presentation 30\% of the bladder cancer patients are MIBC.}
}

\newglossaryentry{NMIBC}
{
    name=None-muscle invasive bladder cancer,
    description={One of the two major types of bladder cancer, at first presentation 70\% of the bladder cancer patients are NMIBC with a 5-year survival rate of 90\%}
}

\newglossaryentry{TCGA}
{
    name=The Cancer Genome Atlas,
    description={A comprehensive project that catalogued genetic mutations responsible for cancer by using genome sequencing and bioinformatics. It represents one of the main sources of tumour information for MIBC}
}

\newglossaryentry{WES}
{
    name=Whole Exon Sequencing,
    description={The sequencing method to obtain mutation information in a sample}
}

\newglossaryentry{RNASEQ}
{
    name=RNA sequecing,
    description={The sequencing method to obtain gene expression information in a sample}
}

\newglossaryentry{GE}
{
    name=Gene expression,
    description={The gene abundance, usually measure in TPMs, which is present in a sample}
}

\newglossaryentry{IFN}
{
    name=Interferon-$\gamma$,
    description={This is a special type of immune response}
}

\newglossaryentry{TF}
{
    name=Transcription factor,
    description={Special genes that regulates the gene expression of other genes}
}

\newglossaryentry{UC}
{
    name=Urothelial Carcinoma,
    description={The most common type of bladder cancer in the Western population, characterised by the malignant transformation of urothelial cells lining the urinary tract}
}

\newglossaryentry{SCC}
{
    name=Squamous Cell Carcinoma,
    description={A less common but notable subtype of bladder cancer, often presenting in mixed morphology alongside urothelial carcinoma}
}


\newglossaryentry{LUMINAL}
{
    name=Luminal,
    description={One of the main subtypes in MIBC which exhibit similar differentiation markers as the healthy bladder. It is usually characterised by more favourable prognostics than basal tumours}
}

\newglossaryentry{BASAL}
{
    name=Basal,
    description={One of the main subtypes of MIBC which exhibits undifferentiated markers. These types of samples have a lower prognosis compared to the luminal tumours}
}



\newglossaryentry{NE}
{
    name=Neuroendocrine-like,
    description={A type of MIBC subgroups which has Neuroendocrine-like expression, usually these samples have the poorest survival prognosis (TCGA, consensus)}
}

\newglossaryentry{MES-LIKE}
{
    name=Mesenchymal,
    description={Similar to Ne-like group but have less immune infiltration}
}

\newglossaryentry{P0}
{
    name=P0,
    description={Freshly isolated cells from stroma infiltration. In this project the gene expression from these samples were used to built the closest molecular representation of the bladder}
}

\newglossaryentry{UD}
{
    name=UD,
    description={Undifferentiated cells following the protocol of Cross et al. (2005). The expression from these cells are used to represent the basal tumours in the non-tumour network representation}
}

\newglossaryentry{ABSCA}
{
    name=ABS-Ca,
    description={re-differentiated the UD cells by changing the growth medium to contain 5\% Adult bovine serum (ABS) and increasing the Ca2+ concentration to physiological levels. These offer a clean tissue state without extrinsic factors as it is in the case of the P0 cells}
}


%%%% Community detection 
\newglossaryentry{SBM}
{
    name=Stochastic Block Model,
    description={A generative model used to identify communities or blocks of nodes in a network}
}

\newglossaryentry{LEIDEN}
{
    name=Leiden,
    description={A community detection method that improves upon the Louvain algorithm by providing more accurate and efficient identification of node communities in a network, belonging to the Modularity Maximisation class}
}

\newglossaryentry{LOUIVAN}
{
    name=Louvain,
    description={A popular community detection method used to identify groups of nodes in a graph, also part of the Modularity Maximisation class, but less efficient and robust compared to the Leiden algorithm}
}


\newglossaryentry{PGCNA}
{
    name=Parsimonious Gene Correlation Network Analysis,
    description={A method developed by \citet{Care2019-ij} to integrate multiple gene expression datasets into a network.}
}

\newglossaryentry{WGCNA}
{
    name=Weighted Gene Correlation Network Analysis,
    description={One of the first methods to formalise the creation and analysis of co-expressed networks. Developed by \citet{Langfelder2008-sn}.}
}


\newglossaryentry{MEV}
{
    name=Module Evaluation Value,
    description={Score introduced by \citep{Care2019-ij} which was adapted and refined to this project's aims. It is used to bridges the gap between the select genes with ModCon and the sample representation.}
}

\newglossaryentry{MODCON}
{
    name=Module Connectivity,
    description={Method used by \citep{Care2019-ij} to select the most relevant genes in a community. This was adapted and used to meet the project's aims}
}

%%%%% Other comuptational
\newglossaryentry{MW}
{
    name=Mannâ€“Whitney U-test,
    description={It is a rank-sum test that tests if two distributions are similar}
}

\newglossaryentry{KW}
{
    name=Kruskal-Wallis H-test,
    description={Extension to Mann-Whitney U-test but it allows comparisons of multiple groups}
}

\newglossaryentry{DEA}
{
    name=Differential Expression Analysis,
    description={A statistical method used to identify genes that are significantly differentially expressed between two groups}
}

\newglossaryentry{GSEA}
{
    name=Gene Set Enrichment Analysis,
    description={A method to determine whether a predefined set of ranked genes is statistically enriched in a specific biological pathway}
}

\newglossaryentry{PI-PLOT}
{
    name=Pi-plot,
    description={It is a score that takes in account the output from DEA by combining the fold-change and adjusted p-value, $q$. This allows plotting four comparison in a 2D scatter plot}
}


\newglossaryentry{PCA}
{
    name=Principal Component Analysis,
    description={Technique to reduce the number of features by projecting them to a lower dimension plane}
}

\newglossaryentry{KMEANS}
{
    name=K-means,
    description={Standard clustering method which groups the points in $K$ clusters by their proximity to the closest centroid}
}

\newglossaryentry{HIERARCHICAL CLUSTERING}
{
    name=Hierarchical clustering,
    description={A clustering method that builds a hierarchy of clusters by either iteratively merging smaller clusters or splitting larger ones to group points based on their similarity}
}