% Exp Setup
\section{Methods}


% Leiden and SBM configuration
The same network pipeline from previous chapter (see \cref{fig:N_I:network_pipeline}) is applied on the gene expression of the MIBC cohort from TCGA. The gene selection strategy for the network is followed as in the cluster analysis chapter, \cref{s:cs:methods}, where genes are considered expressed in at least 90\% of the samples. The top 5000 most varied genes are selected by the median/standard deviation to build the network. This is a larger number (compared to 3500) of genes was used to building a graph to have a wider representation of the datasets. Out of the 5000 genes used for the network 325 are Human Transcription Factors based on \citet{Lambert2018-el}.
 
% Community detection
\subsection*{Community detection} \label{s:N_I:sel_tfs_methods_comm_detection}

This project uses and compares two community detection methods: Leiden and \acrfull{dc-sbm}. The former is a deterministic algorithm, and it was run using the Modularity Maximisation cost function and iterated ten times in each run across the experiment. The \acrshort{dc-sbm} uses a probabilistic approach to find the blocks; for the experiments in this section it was run with 700 iterations ($n\_iter=700$), the entire graph was swept up to 10 times ($mc\_iter=10$ sweeps), and the distribution of the data was set to real exponential\footnote{As described in the \href{https://graph-tool.skewed.de/static/doc/demos/inference/inference.html}{Graph-tool documentation}, the distribution of the edge covariance can be specified given the bounds of the data, as seen in the referenced guide.} ($distribution=real\_exponential$). The number of iterations and sweeps ensured that a detailed search for the communities was performed while keeping the computational time relatively low. The 'real-exponential' parameter is used for edge weights that range from $[0, \infty]$, which is the case for the Spearman correlation values.

% Metrics
The performance of the Leiden algorithm is assessed by Modularity Maximisation (described in \cref{s:lit:mod_max}), which measures the separation between communities; higher values indicate better performance. SBM performance is measured by the \acrfull{mdl}, which assesses the minimum information needed to describe the data; lower values indicate better performance. The number of communities found by the two community detection algorithms is also used as a heuristic to assess performance; it is generally assumed that more communities indicate better performance \citep{Care2019-ij}.

% Mention that there was no standard metric
At the time of running this set of experiments, no standard metric was available to compare the performance of the two methods aside from observing trends in the metrics and changes in the number of groups found. However, in the summer of 2023, \cite{Peixoto2023-se}, Tiago Peixoto published research that adapted \acrshort{mdl} to Modularity models (like Leiden).

%Choosing the Control TF
\subsection*{Control}

To account for the random effects of the TF in the network, there are ten control experiments with 325 non-TF randomly selected and allowed a minimum degree from 3-15. The lower bound is given by the value used for the standard genes, while the higher limit was decided empirically as all the prioritised subsets are used for stratification (see \cref{fig:N_I:sel_tfs}); i.e. selected by ModCon based on their network importance. There was no benefit to increasing the upper bound to values $>$15 as all the TFs were selected by ModCon score across the communities, and there were no gains for increasing it further. This set of experiments led to identifying a subset of TF with high biological significance, as discussed in \cref{s:N_I:sel_tfs,s:N_I:sel_tfs_bio}.

\subsection*{Datasets}

The experiments performed in this section are using the RNAseq gene expression from TCGA's MIBC cohort and the transcription factor list was taken from \citet{Lambert2018-el}. As before, the Kallisto method was used to align the RNA-seq reads using genome version GRCh38 with Gencode annotation version 42. 


