\thispagestyle{plain}
\begin{center}
    \Large        
    \textbf{Abstract}
    \vspace{0.9cm}
\end{center}


% The aim of the project
Traditional stratification methods for muscle-invasive bladder cancer (MIBC) often rely on a single omics approach to derive subtypes, such as gene expression or mutation profiles. This project introduces a novel co-expressed network method that allows the integration of multiple data types at different network levels, aiming to improve MIBC stratification. 

%%%%%% Cluster analysis %%%%%%
% The first chapter of the thesis employs aggressive gene filtering, retaining genes that are highly expressed in approximately 90\% of TCGA samples, along with a simple cluster analysis.
Custer analysis was performed on the gene expression of TCGA's MIBC cohort to establish the starting point in this project. Two of the five identified groups were Basal groups with heterogeneous Interferon-$\gamma$ (IFN-$\gamma$) responses. This is consistent with the \textit{in vitro} research at JBU, which was used to divide the Basal into three subgroups further. The patients with the lowest IFN-$\gamma$ response have the poorest survival prognosis, showing the potential to identify new groups.


%%%%%% Network I %%%%%%
% Novelty: 
% 
% Selective edge pruning - finding 98 TF, which is one of the lowest survival Basal group
The integrative network approach prioritises transcription factors (TF) through selective edge pruning and integrates the mutation burden via edge weights. A subset of 98 TF is found to play an important role in tissue differentiation, and their expression was used to stratify the MIBC. This revealed a Basal group of 20 samples with one of the lowest survival rates found in the literature. The group exhibits squamous markers, has a low immune response, and has statistically significant expressed genes involved in tumour aggressiveness, or that can be treatment targets.


%%%%%% Network II %%%%%%
% Novelty
%
% highly connected communities and the highly connected genes of 122
The network approach is refined and used to build a healthy co-expressed network, which is then used to stratify MIBC. The method identified communities that can be traced to splitting the healthy dataset into new groups, revealing new insights into bladder biology. Mutation integration revealed strongly and highly correlated genes with various biological roles, such as cell proliferation or chromatin remodelling.


%%%%%% Conclusion %%%%%%
% The data integration occurred at several stages, the mutation burden at the weight's level where the TF at the node's degree and expression of both the non-tumour and tumour at the disease stratification.
The integrative co-expressed network approach identified distinct gene subsets and new MIBC groups, underscoring the power of networks as a bioinformatics tool for understanding bladder biology.


\newpage