\chapter{Discussion} \label{s:discussion}

\vspace{3mm}
% \noindent\rule{17cm}{0.2pt}
\fbox {
    \parbox{\linewidth}{
      \begin{itemize}
        \item Reviewing the project aims
        \item Challenges
        \item Contribution to MIBC research
        \item Limitations \& Future work
      \end{itemize}
    }
}
\vspace{3mm}


\section{Meeting the aims}

This PhD project had three aims: 1) develop an integrative method for multi-omics data, 2) use the healthy gene expression to inform tumour stratification, and 3) enable researchers to trace the biological mechanism behind each subtype.

% Summary of the method
The \textbf{iNet} pipeline, introduced in \cref{s:N_I} and further refined in \cref{s:N_II}, is a network-based approach that integrates multiple data types. It incorporates 1) \textbf{edge weight} modifiers to account for mutation burden, 2) selective edge pruning to allow \acrlong{tf} to have a higher \textbf{degree}, and 3) gene expression data from healthy samples to construct a co-expressed \textbf{network}.

% % Community detection - traceability
% Community detection plays an essential role in analysing the co-expressed network as it finds groups of genes, which are then used to stratify the MIBC. 

The project began with a clustering analysis of gene expression data from TCGA's MIBC cohort using aggressive filtering (\cref{s:clustering_analysis}). This approach uncovered heterogeneity in the \acrlong{ifn} response within previously classified basal tumours, later confirmed by $in$-$vitro$ experiments at JBU \citep{Baker2022-bj}. Following this, the project introduced an integrative network approach to both tumour and non-tumour datasets (\cref{s:N_I}), demonstrating that data integration affects network metrics, community detection, and MIBC subtyping. \Cref{s:N_I:sel_pruning} focused on selective edge pruning as one of the integration strategies, identifying 98 TF with a significant impact on co-expressed networks. The tumour expression of these TF revealed a basal group with one of the lowest survival rates observed in the literature. In the final chapter, a refined network approach, iNet, was developed to address the limitations identified in \cref{s:N_I} and enhance data integration. This improved pipeline identified 122 highly mutated genes out of the 5000 genes used that are also strongly co-expressed. From these genes there are several with biological function in both healthy  and cancerous bladder.

% Effects on the network show that the data was integrated successfully computationally
The co-expressed networks built from the non-tumour and tumour datasets differed in network metrics, the number of communities, and the MIBC subgroups derived. Through community detection, traceability is achieved by finding groups of nodes that were shown to link MIBC divisions to some of the communities, work which was covered in more detail in \cref{s:N_II:std_net}.

% 
To conclude, the network method proposed allows the integration of multiple data types, enables traceability, and can be used to build the non-tumour gene expression to inform the MIBC stratification. Novel biological insights were identified at the different stages of the pipeline which implications are covered in more depth below.

% Challengs of the work
\section{Challenges}

The goal of finding subtypes in a disease is a \textit{divide et immpera} approach where the primary problem is divided into smaller solvable parts. The central idea is that the sub-components are more accessible to tackle, and solving them will solve the larger problem. In our case, the problem is the MIBC, and forming a molecular representation of the subtypes helps advance our understanding of the disease and focus efforts on treatable parts. However, dividing a disease into unknown parts comes with many challenges. 

% Challenge of unsupervised learning
One of the primary challenges in our project was assessing the effectiveness of the refinements in computational methods, particularly in unsupervised learning applications. Unsupervised learning, a type of machine learning that discovers patterns in data without the need for labelled responses, presented unique challenges in our research. We utilised clustering metrics such as the Silhouette score, which does not rely on ground truth, to gauge the impact of the changes in the network pipeline and network metrics such as degree and closeness. While these tools helped us understand how method modifications influenced the graphs and MIBC subgroups, they did not reveal whether the network changes had a biological impact. To address this, we relied on our domain knowledge and conducted additional analysis.


% Closest to ground truth 
The closest approximations to ground truth were the previous classifications from TCGA, Lund, and the consensus. However, it was recognised that these classifications are incomplete, as the subtypes have not yet translated into improved clinical outcomes, and they were derived only from gene expression, except for the Lund classifier, which also used \acrlong{ihc} data. While these classifications served as a method to check for MIBC trends identified by various changes in the network pipeline, they fall short of providing more information when new subgroups are discovered, such as the High IFNG or Basal 5 subtypes. The only exception was the work on \gls{IFN} from our lab \citep{Baker2022-bj}, which, in addition to helping with MIBC splits, provided
$in-vitro$ experimental data from a specific chemical perturbation that could be used to validate new groups.

% Analyses used 
The high heterogeneity of bladder tumour added extra complexity to the analysis, and to understand the biological significance, methods such as \acrlong{dea}, \acrlong{gsea}, and Kaplan-Meier survival analysis were employed. This validation process often led to new genes that revealed novel biological insights, contributing to a list of potential new markers. For example, a subset of the 98 TF was found significantly expressed in the Basal/Luminal as well as in the differentiated and undifferentiated bladder tissue, summarised in \cref{tab:N_I:markers_diff,tab:N_I:genes_lum_basal}, focusing the research on regulatory genes in the bladder.


% Impact of the development
In summary, measuring the success of methodological changes required extensive analysis to determine if they had a biological impact. While this process advances the understanding of bladder tissue (both cancerous and non-cancerous), it slows the development of methods and makes MIBC stratification a challenging problem to research.

% Contributon to Knowledge
\section{Advancing MIBC research}

% Basal
\subsubsection*{Basal and \acrlong{ifn} heterogeneity}

Using a standard clustering approach on TCGA's MIBC cohort, the research presented in \cref{s:clustering_analysis} found six subgroups, three of which were previously classified as basal and had different \acrshort{ifn} responses. Of particular interest were the Low IFNG and High IFNG, the former having a low \acrshort{ifn} response and poor 5-year survival, and the latter having a high immune response with more favourable prognosis. This initial research in the project not only encouraged the later development but also highlighted that there is an understudied group of samples that can be potentially targeted by immunotherapy such as VISTA\citep{Baker2022-bj}, thus improving treatments.


% Selective edge pruning
\subsubsection*{A subset of 98 regulatory genes}

% Selective edge pruning
Using the selective edge pruning and the control networks, it was found that there is a subset of 98 TF which have a strong correlation with other genes; \cref{s:N_I:sel_pruning}. The tumour gene expression of the 98 TF to stratified the MIBC cohort into five subtypes, from which Basal 5 is a new group with less than a 20\% chance of survival after 2 years, this being the poorest prognosis seen in the thesis and possibly in the literature. Basal 3 was another basal group with a poor prognosis, which contained samples previously classified as \gls{MES-LIKE} \citep{Marzouka2018-ge}.

% Basla 5
The Basal 5 subtype consists of 20 samples ($\sim$4\% of the cohort) previously classified as basal \citep{Kamoun2020-tj,Robertson2017-mg} or UroB \citep{Marzouka2018-ge}\footnote{The latter having poor survival prognosis} and that none of the samples in the Basal 5 were previously classified as \acrlong{ne}\footnote{\acrlong{ne} have the worst poor survival in the consensus and TCGA classifications \citep{Kamoun2020-tj,Robertson2017-mg}}. The group exhibits squamous markers and has a low immune response, and some of the significantly expressed genes are responsible for the aggressiveness of the tumours, while others can be targeted to improve treatment response.

% Selective edge pruning + controls as a method to find TF
The biological findings indicate that the selective edge pruning and the control networks can potentially represent a new method to find relevant \gls{TF}s in a gene expression dataset. The technique can help the researchers reduce the list of potential gene regulatory networks, focusing on the co-expressed ones.

% High degree
\subsubsection*{Subset of genes with high degree}

The sigmoid weight modifier (or "Reward v2") used in the non-tumour network representation led to a selection of 122 genes with the following properties: 1) high mutation burden, 2) co-expressed with other genes, and 3) their Spearman correlation values are strong (\(>\)0.5); see \cref{s:N_II}. All the genes in this subset had a high degree value and were grouped into small-sized communities with less than 10 nodes. Most of the genes have recognisable biological functions in the bladder, cancerous or non-cancerous. However, there are a few instances like \textit{DIAPH2} that have not been studied in bladder cancer showing the potential of new biological insights.

It is remarkable that the network approach, including the \acrshort{hsbm}, can isolate genes 122 genes from 5000 with a functional role into small communities of more than 10 nodes. An implication is that networks with more genes can be used in the network approach. This is important as gene filtering is usually required before constructing the network; thus, biases or assumptions are imposed on the network before the analysis.

% unknown genes
\subsubsection*{Ensembl genes}

Analysing the networks built from all the non-tumour samples revealed that several communities of Ensembl genes contributed to the ABS-Ca and P0 splits and the division of TCGA's MIBC cohort. The Ensembl genes are only found using the non-tumour dataset (healthy), indicating that many poorly described genes may play an important role in bladder biology. The added complexity is not captured by tumour expression-based classifications alone, and the analysis provides an updated view of the molecular subtypes of MIBC from a healthy perspective. Additionally, despite their lack of protein-coding function, recent research suggests that \acrlong{lncRNA} are implicated in gene regulatory mechanisms and chromatin remodelling \citep{Statello2021-md}, biological processes which are relevant to bladder cancer. This underscores the utility of a network-based approach in identifying lncRNAs, offering additional insight into bladder cancer research.


\subsubsection*{Gene filtering} \label{s:discussion:gene_filt}

The goal stratifying the \acrshort{mibc} is to identify a group of genes specific to each subtype that treatments can then target. The analysis starts with $\sim$33,000 genes from which the unexpressed genes are removed and then a selection of the highest varied genes are clustered by different techniques.

The canonical approach to remove unexpressed genes is to apply a permissive gene filtering, retaining the genes that are expressed in at least 10\% of the samples (4 tumours of 408), allowing for high variance. Conversely, the aggressive gene filtering adopted in this project retains genes expressed in at least 90\% of the samples (367 of the 408 tumours), thus preserving a 'core' subset of genes across the \gls{MIBC} subtypes.

Permissive filtering is more commonly used in disease subtyping, as it retains genes with high variance, which may only be expressed in a specific group of samples. However, as demonstrated in this project, using aggressive gene filtering leads to the identification of new Basal divisions that were previously found by using \acrshort{ihc} data in the Lund cohort \citep{Marzouka2018-ge} or focused studies on the \acrshort{ifn} response in the urothelium.

This approach challenges, but also complements, the canonical methods for filtering expressed genes by demonstrating that an alternative strategy can lead to discovering new disease subtypes. In particular, the Basal subgroups on the \acrshort{ifn} response spectrum exhibit new biological differences and align with other research from \citep{Marzouka2018-ge,Baker2022-bj}.


% Nework for stratifications
\subsubsection*{Network as a tool for MIBC subtyping}

Across the different biological processes of a tissue, the genes do not work in isolation but in relation to each other. Networks can represent this relation and even model it based on different data types. For these reasons, this project used the networks to inform the MIBC stratification. 

Based on the analysis performed in this project, it can be seen that the network representation is a useful and powerful tool for research. Through the network analysis, it was found the 98 TF led to find a subgroup of 20 samples with one of the poorest survival seen and other groups similar to the ones from Lund \citep{Marzouka2018-ge}. Using the sigmoid reward modifier, the network pipeline selected a subset of 122 genes with important molecular features (high, strong correlation and mutated) and various roles in the bladder such as cell proliferation (\textit{LAMA3, LAMA4}) or 

% Limitations
\section{Limitations} \label{s:limitations}

The network approach suffered from a few limitations, which are addressed below. Addressing them will be part of future work to refine the methods.

% Datasets
\subsubsection*{Datasets}

The TCGA and non-tumour datasets are invaluable assets to bladder research, but both samples have some aspects that need consideration. The MIBC cohort from TCGA is comprised of biopsies that are considered to be predominantly aggressive forms of MIBC and come mostly from males (\(\sim\)74\%). and are predominantly white (\(\sim\)80\%).

% Talk
The samples from the non-tumour datasets consist mainly of tissues from female donors (\(\sim\)90\%), some of which are paediatric. However, the TCGA's MIBC cohort is a male-dominant dataset. While in the analysis performed, no biological sex difference was observed, not even in one of the ABS-Ca or P0 splits, the differences in the biological sexes might represent a limitation.

Another difference between the two datasets is that the stroma cells are removed in the non-tumour samples to form a cleaner representation of the bladder tissue. In addition, the samples from the non-tumour dataset do not include \acrlong{ifn} response. However, among the previous MIBC stratification, the stroma infiltration characterises groups or, as it was shown \acrshort{ifn} response. This means that the non-tumour network representation cannot find stroma-like MIBC or high immune infiltration subgroups.


\subsubsection*{ModCon and MEV}

One limitation of the network approach is that ModCon does not account for community size imbalance. This issue was most noticeable in the MIBC subtypes derived from the network, where there are small communities of highly connected nodes; see \cref{s:N_II:high_conn}. ModCon selects 100 genes regardless of the community size, and the MEV score is calculated as the sum of the z-scores of the genes selected by ModCon.

% Implications
When a community size is less than 100 nodes, the MEV score of the smaller community (\(<\)100 nodes) is inherently smaller than that of larger communities. Consequently, smaller communities containing important genes may have a lesser impact on the MIBC stratification.


\subsubsection*{Influences of gene selection}

As mentioned in the previous section, aggressive gene filtering identified basal groups with \acrshort{ifn} heterogeneous responses. This indicates that choosing which genes to include in a network has a \textbf{substantial} impact on the subsequent MIBC stratification stage.

The results in this project indicates that aggressive gene filtering might be more appropriate for constructing a non-tumour network representation, which is then used to stratify the MIBC. Selecting genes expressed in at least 90\% of the non-tumour samples ensures that the genes forming the 'core' of the urothelium are utilised to inform cancer subtyping. This is particularly crucial when integrating the mutation burden, as the gene selection process focuses on disruptions in the co-expression of core genes expressed in the bladder.

Another limitation of the co-expressed network built in this project is that only positive correlations are utilised. This means the network pipeline is constrained to identifying co-expressed genes without capturing downregulated mechanisms. To address this, separate networks can be created where the sign of the Spearman correlation values is inverted, thereby avoiding the challenges associated with using negative weights in a network—since some algorithms do not perform well with negative edge weights.


% Future work
\section{Future work} \label{s:future_work}

Due to the limited time in a PhD project and the unsupervised nature of MIBC subtyping, several areas and limitations could be improved upon and suggested for future work.

\subsubsection*{Larger Networks}

% Discussing the potential of using more genes
% This can go to future work
A finding while performing the research on refined network was that the community detection algorithm (hSBM) could isolate 122 genes from 5000 into small groups of fewer than 10 nodes. These genes were not a computing artefact but exhibited biological properties and functions relevant to bladder cancer. This suggests that the network approach can identify small communities, indicating that more genes can be incorporated into the network. Consequently, the initial assumptions regarding gene filtering can be relaxed, potentially leading to better data representation.

The advantage of not imposing a strict filtering is that it reduces the likelihood of missing important genes. For example, work in the JBU lab identified \textit{RARG} as playing a substantial role in tissue differentiation, but it was ranked around the mid-5000s in the gene selection used throughout the project. A less restrictive input dataset comprising 7,500 of 10,000 genes, can be employed in the next iteration of the network pipeline which will include genes like \textit{RARG}.

\subsubsection*{Higher Mutation Resolution}

Unpublished research in the \acrlong{jbu} lab has observed that some mutated genes are not expressed and may not impact the tumour tissue. As seen in the work with the refined network, the highly connected genes have to satisfy several requirements, discarding confounding factors such as unexpressed mutated genes.

Throughout the project, mutation burden was used to indicate the anomalies affecting genes across the MIBC cohort. This approach oversimplifies the complexity of mutations, as additional information is available: frameshift, nonsense, missense, etc. The existing edge weight modifiers can be adapted to support these data types by increasing/decreasing the Sigmoid shape based on point mutation. Additional information, such as copy number alteration (CNA) or structural variation (SV), could be integrated into the network to represent the changes in bladder cancer further. This could be achieved by adapting the edge pruning method to reflect some of these alterations.

\subsubsection*{Beyond mutations and TF}

The project used mutation burden and TF to model the number of edges and their strength of a node. These data types were chosen for their availability and their importance to bladder cancer, but other data types can be integrated. Disruption in the epigenetic mechanism is another molecular characteristic of bladder cancer which could be integrated into a network by adapting the edge weight modifiers to include data from ATAC-seq.


\subsubsection*{Addressing Community Size Imbalance}

The community size imbalance impacts gene selection through ModCon and indirectly affects the iMEV scores used to stratify the MIBC. This issue can be addressed by selecting a proportional number of genes relative to the community size or by employing a normalisation strategy. Another improvement needed for iMEV is to account for the mutation count of each gene in each sample, as well as the type of mutation. This can be implemented by adding an extra argument to the iMEV equation (\cref{eq:N_II:i_z_score}) to weight both the mutation count and type in the tumour sample.


\subsubsection*{Alternative Correlation Measures}

Other correlation measures in the literature may offer improvements to the network representation. For example, partial correlation removes the influence of other variables on the compared pair, resulting in a 'purer' correlation value \citep{De_la_Fuente2004-ts}. Combining this with selective edge pruning may refine the edges retained for building the network. Reducing the number of edges present in the network will allow for a cleaner integration of the signal introduced by the categorical data from CNV or SV.

\subsubsection*{Gene Filtering Strategies}

This chapter presented a detailed analysis of the effects of two gene filtering strategies and posited that both have merits in uncovering new biology. Therefore, it is necessary to better understand their impact on MIBC stratification and the network approach, particularly concerning graph metrics and community detection methods. 

\subsubsection*{Wobble genes}

The SBM models are probabilistic, and by attempting to optimise for the best community configurations, each node is assigned a probability indicating the likelihood of belonging to a particular community at the end of the runs. This can be viewed as a membership score, highlighting which genes are at the 'boundary' between communities. While the stability of community detection has been a focus throughout this thesis, studying the genes that exhibit instability could also provide new insights, an aspect that warrants further investigation.

\subsubsection*{Software and Other Applications}

The software developed in this project was designed from the outset to apply to other diseases. One of the next steps will be to test it on other gene expression datasets to evaluate the method and the software package in relation with other cancers. In addition, it is needed to perform a comparison with other integrative network approaches such as \citet{Hofree2013-ld,He2017-dj} which used propagation methods to combine somatic mutation to individual gene expression networks.


% Final remakrs
\section{Concluding remarks}

% The Novelty of a Network Approach
The network representation was chosen for this project because of its ability to model relationships between genes. The correlation of gene expression from either tumour or healthy datasets was used to build various network representations, which were, in turn, employed for MIBC stratification.


% Recap of the discovery
The network analysis identified new MIBC subgroups and subsets of genes that have proven to play functional and relevant roles in bladder biology. A subset of 98 regulatory genes was identified by prioritising transcription factors, leading to the discovery of new MIBC groups, including one with a poor survival prognosis. Integrating mutation burden led to the discovery of 122 genes that are highly correlated, strongly associated, and have a high mutation burden. Additionally, using the non-tumour representation led to identifying several lncRNAs that may yield exciting new biological insights.

% 
The strength of the network approach is exemplified by the discovery of new Basal subgroups, mainly those identified using the 98 transcription factors (\cref{s:N_I:sel_pruning}) and the refined network in \cref{s:N_II}. The former narrows the focus to a subset of regulatory genes, while the latter exhibits significantly expressed lncRNAs, which may play an essential role in these groups. These findings indicate the presence of a patient subgroup with more aggressive basal tumours and poor survival prognosis, paving the way for future exploration of potential new treatment targets.

Throughout this thesis, it has been demonstrated that the network approach not only identifies new MIBC subgroups but also highlights gene subsets from data integration. This makes the method not just a tool for subgroup discovery but also a means to gain new biological insights through the integration of multiple data types. The network approach, therefore, becomes a powerful and reliable tool for discovering and understanding new MIBC subgroups.