
\chapter{Introduction}

This chapter covers the motivation of the project, aims and objectives as well as the chapter outline.

\section*{Motivation}

% general intro about tech and improvement and healthcare
In recent human history, there has been tremendous progress in the sciences, and this has been closely linked to the increase in life expectancy. About 150 years ago, the average person lived to about 30 years, but today, we expect an average lifespan of 70 years, with even better prospects in high-income countries like the UK or Japan; see \cref{fig:life_expectancy}. While most technologies have contributed to the improved welfare of the global population, advances in healthcare and our biological understanding have directly enhanced life expectancy. Since cancer is typically a disease associated with ageing, the consequence of a longer-living global population is that more people are now affected by cancer.

% Cancer
% impact of cancer to the world
In an article titled "Is the world making progress against cancer?" \citep{World_in_Data_undated-gc}, Dr. Max Roser provides an overview of humanity's progress in combating the disease. As both the global population and life expectancy rise, it is expected that more people will be predisposed to cancer, which is illustrated by the green line in Figure \ref{fig:cancer_death}; the cancer death rate increased by 66\% in 2017 compared to 1990. The red line shows the cancer death rate per 100,000 people, which increased by only 17\%, indicating that if the world's population had remained at the 1990 level, cancer deaths would have increased by only a quarter. Furthermore, when age is standardized, the death rate actually drops by 15\%. Therefore, \cref{fig:cancer_death} suggests that while there has been some progress in the fight against cancer, greater efforts are needed as life expectancy continues to rise.

% Bladder cancer
According to Cancer Research UK\cite{Cancer_Research_UK2015-cf}, bladder cancer is the 10th leading cause of cancer death, accounting for 3\% of all cancer-related deaths in the UK, with 56\% of cases occurring in people older than 75. Almost half (52.6\%) of those diagnosed with bladder cancer survive for more than five years, and 46.4\% survive for more than ten years.

% Introduce the MIBC
There are two major types of bladder cancer: \acrfull{nmibc} and \acrfull{mibc}. The former has a 5-year survival rate of around 90\% but a high recurrence rate, which impacts the quality of life, while the latter is more aggressive, with a higher chance of becoming metastatic and a worse 5-year survival rate of approximately 50\%. This project focuses on studying \acrlong{mibc}.

% Limitations with MIBC classifications
The current MIBC classifications \citep{Kamoun2020-tj,Robertson2017-mg,Marzouka2018-ge} were derived solely from tumour gene expression data. While some clinical trials \citep{Griffin2024-zr} are underway, these subgroups have yet to significantly improve clinical outcomes. The complex molecular characteristics of MIBC necessitate the development of subtypes based on multi-omics data integration.

% What we're doing
Therefore, the motivation of this project is to address the limitations of current MIBC stratification by developing a new integrative network approach to stratify \acrlong{mibc} in a way that is translatable to the clinic.


\section*{Aims \& Objective}

The underlying hypothesis is that integrating multiple data types, such as somatic mutations or Transcription Factors (TFs), will yield clearer subtypes. Additionally, it is hypothesised that the non-cancerous dataset can reveal new insights into tumour subtyping. Thus, the project has the following aims:
\begin{enumerate}
    \item Create a method that allows the integration of multiple data types.
    \item Enable researchers to trace the biological mechanisms behind each subtype.
    \item Utilise non-cancerous gene expression to inform tumour classifications.
\end{enumerate}

The objectives, or requirements, of the integrative network approach, iNet, are:

\begin{enumerate}
    \item Computational simplicityâ€”complexity should arise from the biology.
    \item Facilitate the identification of genes contributing to different subtypes.
    \item Develop a platform that enables the integration of additional data types beyond those considered in this PhD project.
    \item Ensure the method is modular and data-agnostic, allowing its application to other diseases.
\end{enumerate}

\section*{Thesis Outline}

\textbf{Background} - This chapter begins by introducing the reader to the current state of research on MIBC and the various classification systems found in the literature. It continues with a presentation of the datasets and methods used in the project, followed by a review of disease stratification methods beyond bladder cancer and gene expression. The final two sections of the chapter cover network methods used in biological applications and community detection methods for identifying subgroups within a network.


\textbf{Cluster Analysis} - This section of the thesis delves deeper into the clustering methods used throughout the project, whether to group gene expression data or network outputs. Through a standard clustering analysis process, we discovered that one of the major MIBC groups, the basal subtype, can be further divided into three subgroups with differing immune responses and survival prognoses. Notably, the subgroup with the lowest immune response has the poorest 5-year survival rate.

\textbf{A Network Pipeline for Subtyping} - This research investigates the integration of data into a network approach through three key steps: 1) modelling edge weights proportional to the genes' mutation burden, 2) selective edge pruning, which allows \acrlong{tf} to have more connections than other genes, and 3) using gene expression from freshly isolated samples to build a network representation. The chapter assesses the impact of each of these data integration strategies on the network's metrics.


\textbf{Selective Edge Pruning} - This chapter extends the previous work with a focus on finding the appropriate configuration for the number of edges assigned to TF genes and comparing the Leiden community detection algorithm with \acrfull{sbm}. From the selective edge pruning experiments, 98 TFs were identified as naturally co-expressed with other genes, even in a control network. Using the tumour gene expression of this subset to stratify the TCGA's MIBC cohort, a Basal subgroup of 30 samples was discovered, characterised by the lowest survival prognosis seen in the literature. Notably, this subgroup does not contain samples previously classified as \acrlong{ne} and exhibits squamous markers. \acrshort{sbm} is preferred for community detection as it identifies more communities and is robust in detecting patterns in random data.


\textbf{Network II} - Building on the previous work, this chapter introduces a more refined network approach where the data integration methods from Chapter 4 are adapted to have a larger impact on the network. The research presented identifies 122 genes (TF and non-TF) concentrated in small communities (\(<10\) nodes) that are highly and strongly correlated, as well as having a high mutation burden. This suggests that the network approach can handle larger network sizes. Through the analysis, it was observed that many of the significantly differentially expressed genes are identified by Ensembl IDs, highlighting the limits of current biological knowledge.


\textbf{Discussion \& Future Work} - This chapter presents the significance and implications of the findings throughout the project and discusses potential avenues for future research.
