\chapter{Cluster analysis reveals basal immune heterogeneity} \label{s:clustering_analysis}
\chaptermark{Cluster analysis}

\vspace{3mm}
% \noindent\rule{17cm}{0.2pt}
\fbox {
    \parbox{\linewidth}{
      \begin{itemize}
        % \item Cluster analysis with K-means  and PCA of the TCGA's MIBC cohort
        % \item Three Basal splits based on the immune response
        % \item Evidence for supporting the the Neuronal-like group
        \item Develop and implement a clustering analysis pipeline
        \item Apply the clustering method just to the RNAseq data from the TCGA's MIBC cohort
        \item Establish the referential MIBC subgroups used throughout the project
      \end{itemize}
    }
}
\vspace{3mm}

% Summary
\section{Introduction}

This section covers the work presented at the International Bladder Cancer Network (IBCN) in 2022, which describes the first attempt in the project to stratify the \acrfull{mibc} cohort from TCGA using only gene expression (TPMs) obtained from RNAseq. The aim of this work is to develop and implement a method to determine MIBC subgroups using standard clustering techniques applied solely to RNAseq data. This can then serve as both a starting point and a referential framework for the project.

The introduced pipeline utilises standard clustering techniques, including K-means for clustering, Principal Component Analysis (PCA) for dimension reduction, and various clustering metrics. Despite relying on standard methods, novel subtypes within the major MIBC basal group were identified, potentially aiding biologists in better understanding the immune responses of basal tumours. This indirectly highlights the opportunity to identify new MIBC subtypes.

The chapter is structured into two main parts: the methods are detailed in \cref{s:cs:methods}, which includes justifications for selecting K-means as the clustering model, determining the number of groups, and applying PCA prior to clustering. The configurations of the resulting pipeline are displayed in \cref{fig:cs:clustering_pipeline}. Kaplan-Meier survival analysis has shown that the derived MIBC subgroups exhibit significantly different survival prognoses, with the Neuronal and Basal groups with low Interferon-$\lambda$ showing the poorest survival outcomes.

The final part of the chapter analyses and validates the derived groups, attributing biological functions using a series of tools: tumour purity scores from \citet{Yoshihara2013-wq}, Interferon-$\gamma$ response \citet{Baker2022-bj}, and other classifications from TCGA, Lund, and consensus \citep{Robertson2017-mg,Marzouka2018-ge,Kamoun2020-tj}. Both Differential Expression Analysis (DEA) and Gene Set Enrichment Analysis (GSEA) are employed to further validate the biological functions associated with the MIBC subgroups.

% Where to find the code
The code implementation of the cluster analysis (i.e., methods) used in this chapter can be found in the \textit{ReClassification\_42} Jupyter Notebook and the \textit{modelling.py} script file. All the plots and the code to determine the biological functions of the derived MIBC subgroups are located in the \textit{Discussion\_42\_v2} notebook. All files are available in the \href{https://github.com/vladUng/Phd_thesis_exp}{GitHub repository} that accompanies this PhD project.


\section{Aims}

The aims of this section are:
\begin{itemize}
    \item Establish a clustering analysis pipeline for determining MIBC subgroups
    \item Identify the clustering configuration for determining the MIBC subtypes from gene expression
    \item Determine the metrics that can be used to evaluate and validate the groups
    \item Develop reusable methods to support future research in the following chapters
\end{itemize}



% Methods
% \import{Sections/ClusteringAnalysis/}{methods.tex}

% Experiments
\import{Sections/ClusteringAnalysis/}{experiments.tex}

% Interpretation
\import{Sections/ClusteringAnalysis/}{results.tex}

% Discussion
\section{Discussion} \label{s:cs:discussion}

% Basal groups
\subsection*{New MIBC groups}

Three novel Basal group proportional with the \acrfull{ifn} response were found in this chapter. This discovery was facilitated by initially employing a  clustering approach and subsequently comparing the results with existing MIBC classifications \citet{Baker2022-bj,Marzouka2018-ge}. Group-specific marker genes were identified using a suite of tools: DEA, pi-plots, GSEA, and other metadata available on TCGA.

% High and Medium IFNG
Both the High and Medium IFNG groups exhibit an \acrshort{ifn} response, highlighted by the expression of the \acrshort{ifn} signature from \citet{Baker2022-bj}. The two differ in that the High IFNG samples display a distinct immunoglobulin response, aligning them more closely with samples classified as Luminal Infiltrated. In contrast, the Low and Medium IFNG groups show stronger expression of basal/squamous subtypes compared to the third Basal group. Beyond the immune response differences, the DEA between Low/Med IFNG reveals few genes that are significantly expressed in the Low IFNG group. Some of these genes, highlighted in the analysis performed in \cref{s:cs:basal_interp}, may require further biological investigation, given the poor survival prognosis associated with Low IFNG.

% Urgency to study it
The fact that Low IFNG does not include Neuronal samples, according to our clustering and previous work, and that it has a survival prognosis as poor as that of the Neuronal group, underscores the novelty of this group and the urgency to study it.

% Neuronal group to study
In the TCGA classification, \citet{Robertson2017-mg} identified a Neuronal-like group comprising 20 samples, but our study found a larger group with 32 samples. The Kaplan-Meier survival analysis and the DEA/GSEA performed in \cref{s:cs:ne_interp} confirm the Neuronal-like nature of these subtypes. Compared to previous classifications, additional genes not included in the NE-like signature indicate tumour aggressiveness.

% Mention the LumP and LumInf 
The other two groups identified in this chapter, Luminal Papillary and Luminal Infiltrated, both display the differentiation markers (e.g., UPK, KRT) associated with these subtypes. The latter is distinguished from the former by exhibiting an immune response. The GSEA for the LumP group did not reveal any notable pathways, whereas for the LumInf group, it reinforced the characteristic immune response.


% Say that the output is in the figure
The results of this chapter are summarised in \cref{fig:cs:overview_K_means_6}, which displays the Kaplan-Meier survival analysis of the six groups at the top, and the Sankey diagram comparing our findings with the TCGA and Lund classifiers \citep{Robertson2017-mg,Marzouka2018-ge}. \Cref{fig:cs:overview_survival} clearly shows that the Low IFNG and Neuronal-like groups have the poorest survival outcomes, while the LumP group exhibits the most favourable survival rates. The genes which were significantly expressed for each of the subtype are displayed in \cref{tab:cs:genes_summaries}.
 
\begin{figure}[!htb]
    \centering
    \begin{subfigure}[!t]{1.0\textwidth}
        \includegraphics[width=\textwidth,keepaspectratio]{Sections/ClusteringAnalysis/Resources/discussion/survival_K_6.png}    
        \caption{Survival Kaplan-Meier}
        \label{fig:cs:overview_survival}
    \end{subfigure}
    \centering
    \begin{subfigure}[!t]{1.0\textwidth}
        \includegraphics[width=\textwidth,keepaspectratio]{Sections/ClusteringAnalysis/Resources/discussion/KMeans_6_comp.png}
        \caption{Sankey plot}
        \label{fig:cs:overview_comp}
    \end{subfigure} 
    \centering
    \caption[Overview of the MIBC subtypes]{Overview of the MIBC subtypes derived from using K-means K=5 and the \acrshort{ifn} Basal stratification form \citet{Baker2022-bj}. The first plot shows the Kaplan-Meier survival analysis of the 6 subgroups find, Ne and Low IFNG having the poorest prognosis. The Sankey at the bottom shows how the subtypes found are compared with the TCGA and Lund classifiers \citep{Robertson2017-mg,Marzouka2018-ge}.} 
    \label{fig:cs:overview_K_means_6}
\end{figure}


% Talk about gene filtering


% Talk about the implications of gene filtering
\subsection*{Aggressive gene filtering}

% The canonical approach is to apply a permissive gene filtering approach retains genes that are unexpressed in more than 10\% of the samples (i.e., 4 tumours), allowing for a highly varied set of genes in the dataset. Conversely, the aggressive gene filtering adopted here retains genes that are expressed in at least 367 of the 408 samples, thus preserving a 'core' subset of genes across the MIBC subtypes. The pre-processed data used for clustering analysis incorporated these 'core' genes, which included immune response signals for the various Basal groups.

This chapter employed an aggressive gene filtering approach by retaining genes expressed in at least 90\% of the samples or 367 tumours. In contrast, the 'standard' method in the field involves removing genes from analysis that are expressed in less than 10\% of samples, making it more permissive. The aggressive filtering approach is an indirect finding of this chapter, as it enabled the clustering analysis to establish novel Basal groups based on their immune response. This was possible as the immune activated genes tend to left out by the permissive filtering with the gene selection by highest variance. 

The variance gene expression of the \acrshort{ifn} signature is relatively low since the immune response is present in a large portion of the basal samples (i.e., High and Med IFNG). When using a permissive gene filtering approach, the \acrshort{ifn} gene signature dominates the most varied genes. However, with aggressive gene filtering, which imposes strict limits on gene expression, these highly varied genes selected through permissive filtering are excluded. As a result, this filtering method reveals the \acrlong{ifn} gamma response, which shows the rank comparison of the two filtering strategies in \cref{fig:ifng_rank_genes}.

% Implication of the finding
Finding the \acrlong{ifn} gene signature in the higher ranks of the aggressive filtering but not in the permissive filtering indicates that both approaches reveal different aspects of the same dataset. The permissive strategy promotes genes with high variance, potentially leading to subgroups with significantly different expression patterns. In contrast, aggressive gene filtering promotes gene signatures that are consistently expressed in a larger group of samples and exhibit less variation across the entire cohort.

\begin{figure}[!htb]
    \centering
    \includegraphics[width=1.0\textwidth,keepaspectratio]{Sections/Gene_Sel/ifng_ranks.png}
    \caption[Std/median ranks of the genes in \acrlong{ifn} signature]{The \acrshort{ifn} gene signature \citep{Baker2022-bj} ranked by standard deviation and median in two processed datasets. The first dataset is processed using permissive gene filtering, while the second uses aggressive gene filtering. The bar plot demonstrates that the gene signature is 'hidden' in lower ranks when applying permissive strategy, whereas the aggressive filtering reveals the \acrshort{ifn} gene signature.}
      \label{fig:ifng_rank_genes}
\end{figure}


The aggressive filtering approach is an indirect finding of this chapter, as it enabled the clustering analysis to establish novel Basal groups based on their immune response. This was possible as the immune activated genes tend to left out by the permissive filtering with the gene selection by highest variance. 

The aggressive filtering approach is also important for the subsequent chapters \cref{s:N_I,s:N_II}, where a network representation of the non-tumour samples is used to inform MIBC stratification. This type of filtering is preferred as it selects the 'core' expressed genes across the samples and allows for the integration of mutation data, mimicking disruptions in the healthy bladder.

\subsection{Chapter contributions}

There are two main takeaways from the MIBC analysis. First, a change in gene filtering can have a significant impact on the subgroups identified, potentially revealing novel subgroups. The analysis shows that the group with the lowest \acrfull{ifn} response has the poorest survival rate and is characterised by squamous markers. Understanding this group and further studying it may lead to more targeted treatments. The three basal groups were identified using previous research on the \acrshort{ifn} response of the Basal group \citep{Marzouka2018-ge,Baker2022-bj}, suggesting that integrating additional data types may provide a more comprehensive molecular representation of bladder cancer.

% Mention the methods that will be useful for the following work
In addition to identifying the novel Low IFNG group and its biological significance, this chapter represents the project's first attempt to stratify MIBC. The methods and visualisation tools developed here will be utilised throughout the project to perform cluster analysis and interpret the new MIBC subgroups introduced in the following chapters.


The work in this chapter was presented at the International Bladder Cancer Network (IBCN) conference from Barcelona in 2022 and the main findings can be summarise in the points below:

\begin{itemize}
    \item Three basal subgroups with heterogeneous \acrfull{ifn} immune response
    \item The basal group with the lowest immune response exhibits the worst 5-year survival rate
    \item The basal samples with a strong \acrshort{ifn} response show a more favourable prognosis and can represent potential treatment targets
    \item An aggressive gene filtering of the expressed genes contributes to the findings of the new Basal 
\end{itemize}